<!doctype html>
<html class="no-js" lang="zxx">
<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Moldemort</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="manifest" href="site.webmanifest">
    <link rel="shortcut icon" type="image/x-icon" href="assets/img/favicon.ico">

    <!-- CSS here -->
    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
    <link rel="stylesheet" href="assets/css/owl.carousel.min.css">
    <link rel="stylesheet" href="assets/css/slicknav.css">
    <link rel="stylesheet" href="assets/css/flaticon.css">
    <link rel="stylesheet" href="assets/css/progressbar_barfiller.css">
    <link rel="stylesheet" href="assets/css/gijgo.css">
    <link rel="stylesheet" href="assets/css/animate.min.css">
    <link rel="stylesheet" href="assets/css/animated-headline.css">
    <link rel="stylesheet" href="assets/css/magnific-popup.css">
    <link rel="stylesheet" href="assets/css/fontawesome-all.min.css">
    <link rel="stylesheet" href="assets/css/themify-icons.css">
    <link rel="stylesheet" href="assets/css/slick.css">
    <link rel="stylesheet" href="assets/css/nice-select.css">
    <link rel="stylesheet" href="assets/css/style.css">
</head>
<body>
    <!-- ? Preloader Start -->
    <div id="preloader-active">
        <div class="preloader d-flex align-items-center justify-content-center">
            <div class="preloader-inner position-relative">
                <div class="preloader-circle"></div>
                <div class="preloader-img pere-text">
                    <img src="assets/img/logo/loder.png" alt="">
                </div>
            </div>
        </div>
    </div>
    <!-- Preloader End -->
    <header>
        <!-- Header Start -->
        <div class="header-area">
            <div class="main-header ">
                <div class="header-bottom  header-sticky">
                    <div class="container-fluid">
                        <div class="row align-items-center">
                            <!-- Logo -->
                            <div class="col-xl-2 col-lg-2">
                                <div class="logo">
                                    <a href="index.html"><img src="assets/img/logo/logo.png" alt=""></a>
                                </div>
                            </div>
                            <div class="col-xl-10 col-lg-10">
                                <div class="menu-wrapper  d-flex align-items-right justify-content-end">
                                    <!-- Main-menu -->
                                    <div class="main-menu d-none d-lg-block">
                                        <nav>
                                            <ul id="navigation">
                                                <li><a href="description.html">Project</a>
                                                    <ul class="submenu">
                                                        <li><a href="description.html" class="nav-link">Description</a></li>
                                                        <li><a href="design.html" class="nav-link">Design</a></li>
                                                        <li><a href="implementation.html" class="nav-link">Implementation</a></li>
                                                        <li><a href="contribution.html" class="nav-link">Contribution</a></li>
                                                        <li><a href="future_directions.html" class="nav-link">Future Directions</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="wetlab.html">Wet Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="wetlab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="fungus_identification.html" class="nav-link">Fungus Identification</a></li>
                                                        <li><a href="experiments.html" class="nav-link">Experiments</a></li>
                                                        <li><a href="proof_of_concept" class="nav-link">Proof of concept</a></li>
                                                        <li><a href="engineering.html" class="nav-link">Engineering</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="drylab.html">Dry Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="drylab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="invitro.html" class="nav-link">In-vitro modeling</a></li>
                                                        <li><a href="mathmodel.html" class="nav-link">Mathematical modeling</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="notebook.html">Journal</a>
                                                    <ul class="submenu">
                                                        <li><a href="notebook.html" class="nav-link">Notebook</a></li>
                                                        <li><a href="protocols.html" class="nav-link">Protocols</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="parts.html">Parts</a>
                                                    <ul class="submenu">
                                                        <li><a href="parts.html" class="nav-link">Overview</a></li>
                                                        <li><a href="basic_parts.html" class="nav-link">Basic Parts</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="safety.html">Safety</a></li>
                                                <li><a href="human_practices.html">Human Practices</a>
                                                    <ul class="submenu">
                                                        <li><a href="human_practices.html" class="nav-link">Integrated Human Practices</a></li>
                                                        <li><a href="education.html" class="nav-link">Education and Communication</a></li>
                                                        <li><a href="sustainability.html" class="nav-link">Sustainability</a></li>
                                                        <li><a href="mycoexpo.html" class="nav-link">MycoExpo</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="human_practices.html">Additional Excellence</a>
                                                    <ul class="submenu">
                                                        <li><a href="inclusivity.html" class="nav-link">Inclusivity</a></li>
                                                        <li><a href="entrepreneurship.html" class="nav-link">Entrepreneurship</a></li>
                                                        <li><a href="branding.html" class="nav-link">Branding and Graphic Design</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="team.html">Team</a>
                                                    <ul class="submenu">
                                                      <li><a href="team.html" class="nav-link">Members</a></li>
                                                      <li><a href="attributions.html" class="nav-link">Attributions</a></li>
                                                      <li><a href="collaborations.html" class="nav-link">Collaborations</a></li>
                                                      <li><a href="sponsors.html" class="nav-link">Sponsors</a></li>
                                                      <li><a href="https://igem.org/2021_Judging_Form?id=3979" class="nav-link">Judging Form</a></li>
                                                    </ul>
                                                </li>
                                            </ul>
                                        </nav>
                                    </div>
                                    <!-- Header-btn -->
                                    <!--<div class="header-right-btn d-none d-lg-block ml-20">
                                        <a href="contact.html" class="btn header-btn">Tap</a>
                                    </div>-->
                                </div>
                            </div>
                            <!-- Mobile Menu -->
                            <div class="col-12">
                                <div class="mobile_menu d-block d-lg-none"></div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Header End -->
    </header>
    <main>
        <!--? Hero Start -->
        <div class="slider-area2">
            <div class="slider-height2 d-flex align-items-center">
                <div class="container">
                    <div class="row">
                        <div class="col-xl-12">
                            <div class="hero-cap hero-cap2 pt-70">
                                <h2>Experiments and Results</h2>
                                <nav aria-label="breadcrumb">
                                    <ol class="breadcrumb">
                                      <li class="breadcrumb-item"><a href="index.html">Description</a></li>
                                      <li class="breadcrumb-item"><a href="#">Design</a></li>
                                      <li class="breadcrumb-item"><a href="#">Implementation</a></li>
                                      <li class="breadcrumb-item"><a href="#">Contribution</a></li>
                                      <li class="breadcrumb-item"><a href="#">Future Directions</a></li>
                                    </ol>
                                </nav>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Hero End -->
        <!--? Content Start -->
        <div class="our-cases-area section-padding30">
            <div class="container">
                <div class="row justify-content-center">
                    <div class="col-xl-10 col-lg-10 ">
                        <!-- Section Tittle -->
                        <div class="section-tittle text-left mb-30">
                          <h2>Experiments</h2>
                          <p class="pl-20 pr-20">Bacterial Chitinase Combo 2 (BC2) [<a class="text-warning" href="http://parts.igem.org/Part:BBa_K3979003">BBa_K3979003</a>] is one of the four recombinant chitinases that we have designed as a part of our project for iGEM-2021. It is a hybrid chitinase engineered by combining the catalytic domain of ChiA [<a class="text-warning" href="http://parts.igem.org/Part:BBa_K3979010">BBa_K3979010</a>] from Amycolatopsis orientalis B-37  and the chitin binding domain of ChiB
                            [<a class="text-warning" href="http://parts.igem.org/Part:BBa_K3979009">BBa_K3979009</a>] from Serratia marcescens QMB1466. The chitinase combo has an approximate size of 1.161 kbp and molecular weight of 43.262 kDa. The approximate PI and extinction coefficient as calculated from ProtParam Tool is 9.47 and 73,590 M<sup>-1</sup> cm<sup>-1</sup> (assuming all cysteine bonds exist).
                          </p>
                          <div class="image-block, text-center">
                           <img src="assets/img/project/BC1_Domain_Structure.png" alt="sample image" class="img-fluid">
                           <p><strong>Figure 1.</strong> Domain Structure of BC2
                         </p>
                         </div>
                          <p class="pl-20 pr-20">ChiA [BBa_K3979010] is a wild-type chitinase derived from Streptomyces orientalis (also known Amycolatopsis orientalis) strain B-37 . The chitinase was codon optimised for ease of expression in E.coli BL21 (DE3) using <a class="text-warning" href="https://boost.jgi.doe.gov/">BOOST</a>. Pf-ChiA is an endochitinase capable of hydrolysing and cleaving the (1-4)-beta-linkages in chitin and Chito dextrins. The enzyme has a size of 975 bp and molecular weight of 36.6715 kDa. The approximate PI and extinction coefficient as calculated from <a class="text-warning" href="https://web.expasy.org/protparam/">ProtParam Tool</a> is 9.39 and 48,150 M-1 cm-1 (assuming all cysteine bonds exist).
                          <p class="pl-20 pr-20">The wet lab work plan associated with BC2 and ST can be broadly divided into three steps:</p>
                      </div>
                  </div>
                  <div class="col-md-4 mt-sm-30">
                  <h3 class="mb-20"></h3>
                    <ul class="unordered-list">
                      <li>Cloning</li>
                      <li>Expression and purification</li>
                    </ul>
                  </div>
                  </div>
              </div>
            </div>
      </div>
      <section class="sample-text-area">
              <div class="container box_1170">
                  <h2 class="text-center"><b>Cloning of Bacterial Combo 2 (BC2)</b></h2>
          <br>
          <p class="pl-20 pr-20">
            We were able to successfully clone BC2 into the pET28a vector. The BC2 construct and corresponding primers were obtained from IDT. Upon receiving the construct and primers, both were dissolved in autoclaved Milli-Q water for a working concentration of 10ng/μL and 100uM respectively, as instructed by the manufacturer. The working stock of the same was 1 ng/μL and 10 μM respectively.
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2_FP.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 2.</strong> BC2 Forward Primer</p>
          </div>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2RP.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 3.</strong> BC2 Reverse Primer</p>
          </div>
          <p class="pl-20 pr-20">
            As the first step, we PCR amplified the BC2 construct using KOD polymerase. The melting temperature of the construct was 50℃. PCR conditions were standardized with an initial denaturation at 95℃ for 3 minutes, followed by 35 cycles of denaturation (95℃ for 20 seconds), primer annealing (50℃ for 20 seconds), and extension (70℃ for 1 minute). The final extension was done at 72℃ for 5 minutes. The reaction volume was set at 50μL. PCR was confirmed by performing a 1% agarose gel electrophoresis. The gel image of the same is shown below.
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2_PCR.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 4.</strong> Agarose gel run showing successful PCR amplification.</p>
          </div>
          <p class="pl-20 pr-20">
            PCR cleanup was then done using the Macherey-Nagel (MN) kit and the corresponding concentration was 61.5 ng/μL, measured using nanodrop.
          </p>
          <p class="pl-20 pr-20">
            PCR amplified product and pET28a vector were kept for restriction digestion overnight. Double digestion was carried out using the enzymes BamHI HF and HindIII HF received from NEB. A 20μL reaction was set up for a 1μg template in a cutsmart buffer. Single digestion of the vector was done using both BamHI HF and HindIII HF as the control. Protocols were obtained from NEB double digest calculator. The incubation was done at 37℃ overnight. Digested products were then run-on agarose gel and the image of the same is shown below (Fig. 5).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Gel01.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 5.</strong> Agarose gel run showing digested product of BC2 & pet28a</p>
          </div>
          <p class="pl-20 pr-20">
           Annotations
           <ul class="ordered-list">
               <li>DD: Double Digested</li>
               <li>SDB: Single Digested with BamH1 HF</li>
               <li>SDH: Single Digested with HindIII HF</li>
           </ul>
         </p>
          <p class="pl-20 pr-20">
            The size of the BC2 band obtained was larger than the expected value. It was observed that the restriction was not successful. Due to the same, experiments were repeated starting from PCR. Added to the reason was the  low concentration of purified PCR products.
          </p>
          <p class="pl-20 pr-20">
            PCR of BC2 was done again with a slight modification, increasing the extension at 70℃ from 60 seconds to 80 seconds. The rest of the reaction conditions and methods were the same as earlier. A 1% agarose gel electrophoresis was performed. The PCR was successful and the gel image is as shown below (Fig. 6).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Gel02.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 6.</strong> Agarose gel run showing successful PCR amplification</p>
          </div>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Gel03.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 7.</strong> Agarose gel run showing digested product of BC2 & pet28a</p>
          </div>
           <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                <li>Ladder: 1 kb NEB Ladder</li>
                <li>BC1 (DD):  BC1 Double Digested</li>
                <li>BC2 (DD): BC2 Double Digested</li>
            </ul>
          </p>
          <p class="pl-20 pr-20">
            Gel elution was carried out from the restricted bands using the MN kit using the protocols mentioned in the kit. Concentrations of the vector and insert were 4.9 ng/μL and 65 ng/μL respectively. Using the eluted products, a ligation reaction was carried out. The reaction was set up for 20 μL with T4 DNA ligase. The reaction mixture contained 50 ng of vector and the calculations were made using an in-silico ligation calculator with the molar ratio of vector to insert as 1:3. It was incubated at 23°C for 1 hour.
          </p>
          <p class="pl-20 pr-20">
            The ligated product was then transformed into competent DH5α cells. We followed a standardized set of protocols for chemical transformation. Along with ligated BC2, digested vector (negative control1) and competent cells (control2)  alone were also kept for transformation and plated. Plates were then incubated at 37°C overnight (Figure 8).
          </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Plates.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 8.</strong> Transformed colonies of (a) cloned BC2 in DH5\(\alpha\) competent cells (b) Control 2: only competent DH5\(\alpha\) cells (c) Control 2: digested pet28a in DH5\(\alpha\) competent cell
          </p>
          </div>
          <p class="pl-20 pr-20">
          To confirm the transformation, a colony PCR was performed using Taq-polymerase. For the BC2 construct cells and negative controls, T7 forward primer and gene-specific reverse primers were used. The melting temperature of the primers was 50°C and the extension time was set at 2 minutes. Colony PCR couldn't fetch proper results and the gel contained only genomic bands (Figure 9).
         </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Gel04.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 9.</strong> Agarose gel run of Colony PCR product
          </p>
          </div>
          <p class="pl-20 pr-20">
            Next, we did the ligation again and did overnight ligation at 16°C.  The ligated vector was then chemically transformed into Dh5-alpha cells (figure 10)
          </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-PlateD.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 10.</strong> Transformed colony of cloned BC2 in DH5α competent cells
          </p>
        </div>
          <p class="pl-20 pr-20">
           The transformed colonies were picked up for digestion confirmation. Plasmid isolation was done using the MN kit, following their protocols. Shown on the right is the gel image of restriction digestion (Figure 11).
          </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Gel05.jpg" alt="sample image" class="img-fluid">
            <p><strong>Figure 11.</strong> Agarose gel run of Digestion confirmation</p>
          </div>
          <p class="pl-20 pr-20">
           Annotations
           <ul class="ordered-list">
               <li>Ladder: NEB 1 Kb ladder</li>
               <li>Double digested pet28a using BamH1 HF and HindIII HF</li>
               <li>Undigested vector</li>
               <li>Double digested cloned BC2 using BamH1 HF and HindIII HF</li>
           </ul>
         </p>
          <p class="pl-20 pr-20">
           The plasmids were also verified using Sanger sequencing (Figure 12)
          </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Sanger.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 12.</strong> Sanger Sequencing result of cloned BC2
          </p>
          </div>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2Cloned Map.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 13.</strong> BC2 cloned in pET28a</p>
          </div>
          <p class="pl-20 pr-20">
           Cloned vector is then transformed into the E. coli BL21(DE3) strain for expression (figure 14)
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-Plates-01.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 14.</strong> Transformed colonies of (a) Empty vector in BL21 cells (b) Cloned BC2 in BL21 cells (c) only BL21 cells</p>
          </div>
          <br>
          <br>
          <h2 class="text-center"><b>Expression and Purification of Bacterial Combo 2 (BC2)</b></h2>
          <br>
          <h3 class="text-center">Protein Expression</h3>
          <p class="pl-20 pr-20">
            The general procedure followed for expression included a first primary inoculation followed by secondary inoculation and then SDS. Transformed BL21(DE3) cells were grown kanamycin-LB-plates and colonies were picked for inoculation. They were inoculated in 10 ml LB media with kanamycin (50 ng/μL) and incubated for 12 hours. It was then added to the secondary culture and IPTG was added once the OD value reaches 0.6. After IPTG addition, the culture was incubated and pellets down. Our first aim was to standardize the amount of IPTG to be added and the incubation time.
          </p>
          <p class="pl-20 pr-20">
           For this, we carried out IPTG induction and expression in different sets of IPTG concentrations and temperatures. Our experimental conditions were as follows:
          </p>
          <p class="pl-20 pr-20">
            <ol class="ordered-list">
                 <li>35°C 1mM IPTG</li>
                 <li>35°C 0.1mM IPTG</li>
                 <li>37°C 0.1mM IPTG</li>
                 <li>37°C 1mM IPTG</li>
                 <li>30°C 0.1mM IPTG</li>
                 <li>30°C 1mM IPTG</li>
                 <li>26°C 1mM IPTG</li>
                 <li>16°C 0.1mM IPTG</li>
                 <li>16°C 1mM IPTG</li>
                 <li>Uninduced primary inoculum</li>
                 <li>26°C 0.1mM Supernatant</li>
                 <li>16°C 1mM Supernatant</li>
            </ol>
          </p>
          <p class="pl-20 pr-20">
            The cells which were pellet down after IPTG induction were lysed and an SDS-PAGE was performed using the lysate (Figure 15). We could understand that our protein was getting expressed in all concentrations of IPTG and temperatures. We could specifically see a broader band in condition number 1, at the required size of 43.262 kDa. Hence, 1mM IPTG and 35℃ were chosen for further experiments downstream.
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS01.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 15.</strong> SDS PAGE result of IPTG induction and expression at different conditions.</p>
          </div>
          <p class="pl-20 pr-20">
            Our next aim was to check whether the proteins are expressed in inclusion bodies or not. For this, the cells were induced using 1mM IPTG and kept for incubation at  35℃ for 5 hrs. Then the cells were  lysed using sonication after adding the lysis buffer (50mM NaPB buffer pH 7, 500mM NaCl, 1mM PMSF, 0.05% beta mercaptoethanol, 5%glycerol, 0.5% triton X-100, 1X protease inhibitor).  After centrifugation, both supernatant and pellets were mixed with the loading dye to perform 10% SDS-PAGE along with uninduced control and induced non-sonicated samples (Figure 16).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS02.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 16.</strong> SDS PAGE result to check whether proteins are expressed in inclusion bodies or not when induced by 1mM IPTG and incubated at 35°C.</p>
          </div>
          <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                 <li>Ladder: NEB Protein standard ladder (11-245 kDa)</li>
                 <li>Uninduced sample (control)</li>
                 <li>Induced sample without lysing the cells</li>
                 <li>Pellet after lysing the induced cells</li>
                 <li>Supernatant after lysing the induced cells</li>
            </ul>
          </p>
           <p class="pl-20 pr-20">
            We could understand that proteins were forming in the inclusion bodies. So, we decided to induce at a lower temperature of 16°C, since inducing the protein at lower temperature may cause the normal folding of proteins by giving it enough time to fold properly.
          </p>
          <p class="pl-20 pr-20">
            Inducing at 16°C followed by lysis using sonication. SDS result shows that protein is forming in cytoplasm as well as inclusion body. (Figure 17)
          </p>
           <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS03.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 17.</strong> SDS PAGE results to check whether proteins are expressed in inclusion bodies or not when induced by 1mM IPTG and incubated at 16°C</p>
          </div>
          <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                 <li>Ladder: NEB Protein Standard ladder (11-245 kDa)</li>
                 <li>Sample from Supernatant (after sonication)</li>
                 <li>Sample from Pellet (after sonication)</li>
                 <li>Sample from Pellet (after sonication)</li>
                 <li>Sample from Supernatant (after sonication)</li>
            </ul>
          </p>
          <p class="pl-20 pr-20">
            The expression of BC2 was thus standardized as mentioned above and our project reached the next goal, purification.
          </p>
          <br>
          <h3 class="text-center">Purification</h3>
          <p class="pl-20 pr-20">
            Secondary inoculum (1 liter) induced with 1mM IPTG and incubated overnight at 16°C was pelleted down at 13,000 RPM for 15 mins. Cells were resuspended in 50 ml lysis buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet) and sonicated in pulse mode (30 seconds ON and 30 seconds OFF for 15 cycles with 55% amplitude). The supernatant was collected and loaded into Ni-NTA column pre-equilibrated  with equilibration buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet). It was then washed using 50 mL wash buffer (50 mM sodium phosphate buffer (pH 7), 0.5 mM NaCl, 1 mM EDTA) and eluted in 30 mL elution buffer (50 mM sodium phosphate buffer (pH 7), 0.5 mM NaCl, 1 mM EDTA, 250 mM Imidazole). Collected samples were analyzed using 13% SDS-PAGE (Figure 18).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS04.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 18.</strong> SDS PAGE result of Ni-NTA.</p>
          </div>
          <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                 <li>F: Flow Through</li>
                 <li>W: Wash Flow Through</li>
                 <li>E: Elution Flow Through</li>
                 <li>1-20: Elution Fractions</li>
            </ul>
          </p>
          <p class="pl-20 pr-20">
            From the gel, it was understood that our protein was coming along with the flowthrough at a band size of 43.262 kDa. We tried elution in NaPB buffer and also tried Ni-NTA from other manufacturers. All these series of experiments suggested that our 6xHis is somehow not able to bind to Ni-NTA, possibly due to its incorporation into protein tertiary structure. After a brief brainstorming session, we decided to perform urea dialysis for purifying BC2.
          </p>
          <p class="pl-20 pr-20">
              For performing urea dialysis, the cells were pelleted down after IPTG induction and lysed using lysis buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet). The pellet was collected, washed twice with wash buffer (50mM, NaPB pH7, 0.5mM NaCl, 0.5% Triton X)  followed by wash with 10mM CaCl2 and then pellets were resuspended in a extraction buffer containing  8M urea. This was done to denature the protein so as the 6X-His tag will get exposed leading to efficient binding of the protein to the column. After urea treatment, the supernatant was collected and loaded on the Ni NTA resin. When the eluted fraction was analyzed using SDS-PAGE, strong bands of the protein were visible. The gel image is shown below: (Figure 19).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS05.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 19.</strong> SDS PAGE result of Ni-NTA.</p>
          </div>
          <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                 <li>F: Flow Through</li>
                 <li>W: Wash Flow Through</li>
                 <li>E: Elution Flow Through</li>
                 <li>1-30: Elution Fractions</li>
            </ul>
          </p>
          <p class="pl-20 pr-20">
              We did stepwise dialysis, the denatured protein solution was first brought to equilibrium with a high denaturant concentration (buffer containing 6M Urea), then, the concentration was decreased and brought to equilibrium at a medium concentration (buffer containing 4M Urea) and, then, further decreased and brought to equilibrium at a low concentration (buffer containing 2M Urea). At last protein, the solution was brought to equilibrium with a buffer containing 0M urea. But the protein was found to be aggregating under prolonged dialysis for removing the urea. Hence, we adopted buffer exchange using protein concentrator columns as a fast method of removing urea.  This method was also not successful in providing protein of good concentrations. To overcome this obstacle, we did some literature survey and found that ethanol precipitation method can be used for proteins that are aggregating during urea dialysis.
          </p>
          <p class="pl-20 pr-20">
              Following literature, a new batch of protein expression was started. The cells were lysed and the pellet was resuspended in NaPB buffer containing 8M urea. The Ni-NTA column was loaded with this solution and the column was eluted. SDS-PAGE was performed for knowing the fractions containing protein bands. The image of the PAGE gel showing protein bands is shown below: (Figure 20).
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/wetlab/BC2-SDS06.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 20.</strong> SDS PAGE result of Ni-NTA.</p>
          </div>
          <p class="pl-20 pr-20">
            Annotations
            <ul class="ordered-list">
                 <li>F: Flow Through</li>
                 <li>W: Wash Flow Throu gh</li>
                 <li>E: Elution Flow Through</li>
                 <li>1-20: Elution Fractions</li>
            </ul>
          </p>
           <p class="pl-20 pr-20">
               To the fractions showing protein bands, we added 100% ice-cold ethanol and incubated the solution for 1 hour in -20 degree Celsius. This was followed by centrifugation for bringing down the precipitated protein. The obtained pellet was washed in 90% ethanol and special care was taken to remove the supernatant to the maximum possible extent. The washed pellet was resuspended in 1X PBS containing 0.1% SDS. While concentrating our protein, we did see aggregation when concentrated below 2 mL. Thus, we changed the composition of the buffer by adding stabilizers such as 10% glycerol and increasing the NaCl concentration to 200 mM [Final buffer composition: 1X PBS, 0.1%SDS, 200mM NaCl, and 10% glycerol]. This method was successful in obtaining good concentrations of protein having a tertiary structure, as was confirmed through CD spectroscopy.
          </p>
        </section>
        <section class="sample-text-area">
                <div class="container box_1170">
                    <h2 class="text-center"><b>Cloning of <i>Streptomyces</i> wild-type chitinase (ST)</b></h2>
            <br>
            <p class="pl-20 pr-20">
              We were able to successfully clone ST into the pET28a vector. The Streptomyces (ST) chitinase construct and corresponding primers were obtained from IDT. Upon receiving the construct and primers, both were dissolved in autoclaved Milli-Q water for a working concentration of 10ng/μL and 100uM respectively, as instructed by the manufacturer. The working stock of the same was 1 ng/μL and 10 uM, respectively.
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/StreptoFP.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 21.</strong> ChiA Forward Primer </p>
            </div>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/StreptoRP.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 22.</strong> ChiA Reverse Primer</p>
            </div>
            <p class="pl-20 pr-20">
              As the first step, we PCR amplified the ST construct using KOD polymerase. The melting temperature of the construct was 50℃. PCR conditions were standardized with an initial denaturation at 95℃ for 3 minutes, followed by 35 cycles of denaturation (95℃ for 20 seconds), primer annealing (50℃ for 20 seconds), and extension (70℃ for 2 minutes). The final extension was done at 72℃ for 5 minutes. The reaction volume was set at 50μL. PCR was confirmed by performing a 1% agarose gel electrophoresis. The gel image of the same is shown below (Fig.23).
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_SDS01.jpg" alt="sample image" class="img-fluid">
              <p><strong>Figure 23.</strong> Agarose gel run showing successful PCR amplification</p>
            </div>
             <p class="pl-20 pr-20">
              Annotations
              <ul class="ordered-list">
                   <li>Ladder: 1 kb NEB Ladder</li>
                   <li>1 & 2 : ST chitinase gene (975 bp)</li>
            </p>
            <p class="pl-20 pr-20">
              PCR cleanup was then done using the Macherey-Nagel (MN) kit, and the corresponding concentration was 102 ng/μL, measured using nanodrop.
            </p>
            <p class="pl-20 pr-20">
              PCR amplified product and pET28a vector were kept for restriction digestion for 6.5 hours. Double digestion was carried out using the enzymes BamHI HF, and HindIII HF received from NEB. A 40μL reaction was set up for a 1μg template in a cutsmart buffer. Double digestion of the vector having BC2 was done using BamHI HF and HindIII HF as the control. Protocols were obtained from NEB double digest calculator. The incubation was done at 37℃ for 6.5 hours. Digested products were then run on agarose gel, and the image of the same is shown below (Figure.24).
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_SDS02.jpg" alt="sample image" class="img-fluid">
              <p><strong>Figure 24.</strong> Agarose gel run showing successful digestion of ST and cloned BC2 with BC2 and pet28a band</p>
            </div>
            <p class="pl-20 pr-20">
              Gel elution was carried out from the restricted bands using the MN kit using the protocols mentioned in the kit. Concentrations of the vector and insert were 17.9 ng/μL and 10.4 ng/μL, respectively. Using the eluted products, a ligation reaction was carried out. The reaction was set up for 20 μL with T4 DNA ligase. The reaction mixture contained 50 ng of vector, and the calculations were made using an in-silico ligation calculator with the molar ratio of vector to insert as 1:3. It was incubated at 23°C for 30 mins.
            </p>
            <p class="pl-20 pr-20">
              The ligated product was then transformed into competent DH5-alpha cells. We followed a standardized set of protocols for chemical transformation. Along with ligated ST, digested vector (negative control1) and competent cells (control2)  alone were also kept for transformation and plated. Plates were then incubated at 37°C overnight.
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST-Plates01.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 25.</strong> LB Plates plated with
              A. DH5alpha E coli. transformed with pet28a containing ST, 8-10 colonies;
              B. DH5alpha E coli. transformed with digested pet28a, no colonies;
              C. DH5alpha E coli. transformed without any vector, no colonies
             </p>
            </div>
            <p class="pl-20 pr-20">
             To confirm the transformation, a colony PCR using Taq-polymerase was performed. Six colonies were picked for which, T7 forward primer and T7 terminator reverse primer were used for amplification. PCR conditions were standardized with an initial denaturation at 95℃ for 10 minutes, followed by 20 cycles of denaturation (95℃ for 30 seconds), primer annealing (50℃ for 30 seconds), and extension (72℃ for 3 minutes). The final extension was done at 72℃ for 10 minutes. The expected band size was around 1.3kb, including the 937 kb gene and the T7 promoter and T7 terminator. Cloned BC2 was used as a positive control.
             </p>
             <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_SDS03.jpg" alt="sample image" class="img-fluid">
              <p><strong>Figure 26.</strong> Colony PCR of ST </p>
            </div>
            <p class="pl-20 pr-20">
              Annotations
              <ul class="ordered-list">
                   <li>Colony PCR product of ST cloned in DH5alpha (1.3 kb)</li>
                   <li>Positive control: Colony PCR product of BC2 cloned in DH5alpha</li>
                   <li>Ladder: NEB 1 Kb Ladder</li>
            </p>
            <p class="pl-20 pr-20">
              Four of the 6 selected colonies were picked up for digestion confirmation. Plasmid isolation was done using the MN kit, following their protocols. Shown below is the gel image of restriction digestion (Figure 27).
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_SDS04.jpg" alt="sample image" class="img-fluid">
              <p><strong>Figure 27.</strong> Restriction digestion confirmation of ST cloned in DH5alpha
             </p>
            </div>
            <p class="pl-20 pr-20">
             Cloned vectors were then transformed into the E. coli BL21(DE3) strain for expression (Figure 6).
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST-Plates02.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 28.</strong> LB Plates plated with
            A. BL21(DE3) E coli. transformed with pet28a containing ST;
            B. BL21(DE3) E coli. transformed with empty pet28a;
            C. BL21(DE3) E coli. transformed without any vector
             </p>
             <div class="image-block, text-center">
               <img src="assets/img/wetlab/StreptoCloned Map.png" alt="sample image" class="img-fluid">
               <p><strong>Figure 29.</strong> ChiA Cloned in pET28a.</p>
             </div>
            <p class="pl-20 pr-20">
             From the transformed plate, four colonies were picked and restriction digestion was done using BamHI and HindIII to confirm the same. Glycerol stocks were made for the positive colonies.
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_SDS05.jpg" alt="sample image" class="img-fluid">
              <p><strong>Figure 30.</strong> Restriction digestion confirmation </p>
            </div>
            <br>
            <h2 class="text-center"><b>Expression and Purification of Streptomyces (ST)</b></h2>
            <br>
            <h3 class="text-center">Protein Expression</h3>
           <p class="pl-20 pr-20">
            The general procedure followed for expression included a first primary inoculation followed by secondary inoculation and then SDS. Transformed BL21(DE3) cells were grown kanamycin-LB-plates and colonies were picked for inoculation. They were inoculated in 10 ml LB media with kanamycin (50 ng/μL) and incubated for 12 hours. It was then added to the secondary culture and IPTG was added once the OD value reaches 0.6. After IPTG addition, the culture was incubated and pellets down. Our first aim was to standardize the amount of IPTG to be added and the incubation time.
            </p>
            <p class="pl-20 pr-20">
            For this, we carried out IPTG induction and expression in different sets of IPTG concentrations and temperatures. We conducted a few sets of trial-and-error experimental conditions, four colonies were picked from which varied from a range of 16℃ to 37℃ and IPTG concentrations of 0.1 mM and 1 mM. From those, we could understand that our protein of interest was getting expressed at 37℃ and 1mM IPTG.
            </p>
            <br>
            <h3 class="text-center">Purification</h3>
            <p class="pl-20 pr-20">
              Secondary inoculum (1 liter) induced with 1mM IPTG and incubated for seven hours at 37°C was pelleted down at 11,000 RPM for 20 mins. Cells were resuspended in 50 ml lysis buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet) and sonicated in pulse mode (30 seconds ON and 30 seconds OFF for 15 cycles with 55% amplitude). The supernatant was collected and loaded into Ni-NTA column pre-equilibrated  with equilibration buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet). It was then washed using 50 mL wash buffer (50 mM sodium phosphate buffer (pH 7), 0.5 mM NaCl, 1 mM EDTA) and eluted in 30 mL elution buffer (50 mM sodium phosphate buffer (pH 7), 0.5 mM NaCl, 1 mM EDTA, 250 mM Imidazole). Collected samples were analyzed using 12% SDS-PAGE (Figure 31).
            </p>
            <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_Purification01.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 31.</strong> Ni-NTA of Lysate (supernatant) of transformed BL21(DE3) with pet28a cloned with ST
            </p>
          </div>
            <p class="pl-20 pr-20">
              From the gel, it was understood that our protein came only in the pellet obtained after lysis and not in fractions collected after Ni-NTA purification. The lysed pellet had protein with a band size of 36.67 kDa.
            </p>
            <p class="pl-20 pr-20">
                Due to lack of time, we couldn’t standardize the purification of native protein via Ni-NTA. We proceeded with urea treatment to expose the His-tag if hidden.  For that, the cells were pelleted down after IPTG induction and lysed using lysis buffer (50 mM sodium phosphate buffer (pH 7), 1 mM PMSF, 0.5 M NaCl, 0.05 % BME, 5 % Glycerol, 0.5 % Triton X-100, 1 protease inhibitor tablet). The pellet was collected, washed twice with a wash buffer followed by a wash with CaCl2 and then pellets were resuspended in an extraction buffer containing  8M urea. This was done to denature the protein so as the 6X-His tag will get exposed, leading to efficient binding of the protein to the column. After urea treatment, the supernatant was collected and loaded on the Ni NTA resin. When the eluted fractions were analyzed using SDS-PAGE, strong protein bands were visible in fractions 1 & 3. The gel image is shown below (Figure 32).
            </p>
             <div class="image-block, text-center">
              <img src="assets/img/wetlab/ST_Purification02.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 32.</strong> Ni-NTA of urea treated Lysate (pellet) of transformed BL21(DE3) with pet28a cloned with ST
             </p>
            </div>
             <p class="pl-20 pr-20">
               To the fractions showing protein bands, we added 100% ice-cold ethanol and incubated the solution for 1 hour in -20 degrees Celsius. This was followed by centrifugation for bringing down the precipitated protein. The obtained pellet was washed in 90% ethanol, and special care was taken to remove the supernatant to the maximum possible extent. The washed pellet was resuspended in 1X PBS containing 0.1% SDS, 200mM NaCl and 10% glycerol.After purifying the protein, we first measured the concentration of the enzyme using nanodrop. To make the protein concentration accurate, we imputed the protein’s dielectric constant and molecular weight (predicted by ProtParam). Then, we concentrated the protein using an amicon filter to 0.749 mg/ml.
             </p>
        </section>
        <!-- About Area End -->
    </main>
    <footer>
        <div class="footer-wrapper">
           <!-- Footer Start-->
           <div class="footer-area footer-padding">
               <div class="container ">
                   <div class="row justify-content-between">
                       <div class="col-xl-4 col-lg-3 col-md-8 col-sm-8">
                           <div class="single-footer-caption mb-50">
                               <div class="single-footer-caption mb-30">
                                   <!-- logo -->
                                   <div class="footer-logo mb-35">
                                       <a href="index.html"><img src="assets/img/logo/logo2_footer.png" alt=""></a>
                                   </div>
                                   <div class="footer-tittle">
                                       <div class="footer-pera">
                                           <p>The iGEM Team of IISER Thiruvananthapuram</p>
                                       </div>
                                   </div>
                                   <!-- social -->
                                   <div class="footer-social">
                                       <a href="https://twitter.com/igem_iisertvm"><i class="fab fa-twitter"></i></a>
                                       <a href="https://instagram.com/igem_iisertvm"><i class="fab fa-instagram"></i></a>
                                       <a href="https://www.youtube.com/channel/UC20hJ-yyGrhZH2PbUrRyI8w"><i class="fab fa-youtube"></i></a>
                                       <a href="https://linktr.ee/igem_iisertvm"><i class="fas fa-link"></i></a>
                                   </div>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-3 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Some Links</h4>
                                   <ul>
                                       <li><a href="#">Info 1</a></li>
                                       <li><a href="#">Info 2</a></li>
                                       <li><a href="#">Info 3</a></li>
                                       <li><a href="#">Info 4</a></li>
                                       <li><a href="#">Info 5</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-2 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>iGEM Team</h4>
                                   <ul>
                                       <li><a href="#">About</a></li>
                                       <li><a href="#">Review</a></li>
                                       <li><a href="#">Info</a></li>
                                       <li><a href="#">Human</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-3 col-lg-4 col-md-6 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Contact us</h4>
                                   <ul>
                                       <li><a href="#">igem@iisertvm.ac.in</a></li>
                                       <li><a href="#">Vithura, IISER TVM</a></li>
                                       <li><a href="#">Maruthamala PO</a></li>
                                       <li class="number"><a href="#">(+91) 8240966465</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                   </div>
               </div>
           </div>
           <!-- footer-bottom area -->
           <!--<div class="footer-bottom-area">
               <div class="container">
                   <div class="footer-border">
                       <div class="row d-flex align-items-center">
                           <div class="col-xl-12 ">
                               <div class="footer-copy-right text-center">
                                  <p>
                                      Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved.
                                  </p>
                                  </div>
                              </div>
                          </div>
                      </div>
                  </div>
              </div>-->
              <!-- Footer End-->
          </div>
      </footer>

      <!-- Scroll Up -->
      <div id="back-top" >
        <a title="Go to Top" href="#"> <i class="fas fa-level-up-alt"></i></a>
    </div>

    <!-- JS here -->

    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3.0.1/es5/tex-mml-chtml.js"></script>
    <script src="./assets/js/vendor/modernizr-3.5.0.min.js"></script>
    <!-- Jquery, Popper, Bootstrap -->
    <script src="./assets/js/vendor/jquery-1.12.4.min.js"></script>
    <script src="./assets/js/popper.min.js"></script>
    <script src="./assets/js/bootstrap.min.js"></script>
    <!-- Jquery Mobile Menu -->
    <script src="./assets/js/jquery.slicknav.min.js"></script>

    <!-- Jquery Slick , Owl-Carousel Plugins -->
    <script src="./assets/js/owl.carousel.min.js"></script>
    <script src="./assets/js/slick.min.js"></script>
    <!-- One Page, Animated-HeadLin -->
    <script src="./assets/js/wow.min.js"></script>
    <script src="./assets/js/animated.headline.js"></script>
    <script src="./assets/js/jquery.magnific-popup.js"></script>

    <!-- Date Picker -->
    <script src="./assets/js/gijgo.min.js"></script>
    <!-- Nice-select, sticky -->
    <script src="./assets/js/jquery.nice-select.min.js"></script>
    <script src="./assets/js/jquery.sticky.js"></script>
    <!-- Progress -->
    <script src="./assets/js/jquery.barfiller.js"></script>

    <!-- counter , waypoint,Hover Direction -->
    <script src="./assets/js/jquery.counterup.min.js"></script>
    <script src="./assets/js/waypoints.min.js"></script>
    <script src="./assets/js/jquery.countdown.min.js"></script>
    <script src="./assets/js/hover-direction-snake.min.js"></script>

    <!-- contact js -->
    <script src="./assets/js/contact.js"></script>
    <script src="./assets/js/jquery.form.js"></script>
    <script src="./assets/js/jquery.validate.min.js"></script>
    <script src="./assets/js/mail-script.js"></script>
    <script src="./assets/js/jquery.ajaxchimp.min.js"></script>

    <!-- Jquery Plugins, main Jquery -->
    <script src="./assets/js/plugins.js"></script>
    <script src="./assets/js/main.js"></script>

</body>
</html>
