<!doctype html>
<html class="no-js" lang="zxx">
<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Moldemort</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="manifest" href="site.webmanifest">
    <link rel="shortcut icon" type="image/x-icon" href="assets/img/favicon.ico">

    <!-- CSS here -->
    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
    <link rel="stylesheet" href="assets/css/owl.carousel.min.css">
    <link rel="stylesheet" href="assets/css/slicknav.css">
    <link rel="stylesheet" href="assets/css/flaticon.css">
    <link rel="stylesheet" href="assets/css/progressbar_barfiller.css">
    <link rel="stylesheet" href="assets/css/gijgo.css">
    <link rel="stylesheet" href="assets/css/animate.min.css">
    <link rel="stylesheet" href="assets/css/animated-headline.css">
    <link rel="stylesheet" href="assets/css/magnific-popup.css">
    <link rel="stylesheet" href="assets/css/fontawesome-all.min.css">
    <link rel="stylesheet" href="assets/css/themify-icons.css">
    <link rel="stylesheet" href="assets/css/slick.css">
    <link rel="stylesheet" href="assets/css/nice-select.css">
    <link rel="stylesheet" href="assets/css/style.css">
</head>
<body>
    <!-- ? Preloader Start -->
    <div id="preloader-active">
        <div class="preloader d-flex align-items-center justify-content-center">
            <div class="preloader-inner position-relative">
                <div class="preloader-circle"></div>
                <div class="preloader-img pere-text">
                    <img src="assets/img/logo/loder.png" alt="">
                </div>
            </div>
        </div>
    </div>
    <!-- Preloader End -->
    <header>
        <!-- Header Start -->
        <div class="header-area">
            <div class="main-header ">
                <div class="header-bottom  header-sticky">
                    <div class="container-fluid">
                        <div class="row align-items-center">
                            <!-- Logo -->
                            <div class="col-xl-2 col-lg-2">
                                <div class="logo">
                                    <a href="index.html"><img src="assets/img/logo/logo.png" alt=""></a>
                                </div>
                            </div>
                            <div class="col-xl-10 col-lg-10">
                                <div class="menu-wrapper  d-flex align-items-right justify-content-end">
                                    <!-- Main-menu -->
                                    <div class="main-menu d-none d-lg-block">
                                        <nav>
                                            <ul id="navigation">
                                                <li><a href="description.html">Project</a>
                                                    <ul class="submenu">
                                                        <li><a href="description.html" class="nav-link">Description</a></li>
                                                        <li><a href="design.html" class="nav-link">Design</a></li>
                                                        <li><a href="implementation.html" class="nav-link">Implementation</a></li>
                                                        <li><a href="contribution.html" class="nav-link">Contribution</a></li>
                                                        <li><a href="future_directions.html" class="nav-link">Future Directions</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="wetlab.html">Wet Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="wetlab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="fungus_identification.html" class="nav-link">Fungus Identification</a></li>
                                                        <li><a href="experiments.html" class="nav-link">Experiments</a></li>
                                                        <li><a href="proof_of_concept" class="nav-link">Proof of concept</a></li>
                                                        <li><a href="engineering.html" class="nav-link">Engineering</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="drylab.html">Dry Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="drylab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="invitro.html" class="nav-link">In-vitro modeling</a></li>
                                                        <li><a href="mathmodel.html" class="nav-link">Mathematical modeling</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="notebook.html">Journal</a>
                                                    <ul class="submenu">
                                                        <li><a href="notebook.html" class="nav-link">Notebook</a></li>
                                                        <li><a href="protocols.html" class="nav-link">Protocols</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="parts.html">Parts</a>
                                                    <ul class="submenu">
                                                        <li><a href="parts.html" class="nav-link">Overview</a></li>
                                                        <li><a href="basic_parts.html" class="nav-link">Basic Parts</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="safety.html">Safety</a></li>
                                                <li><a href="human_practices.html">Human Practices</a>
                                                    <ul class="submenu">
                                                        <li><a href="human_practices.html" class="nav-link">Integrated Human Practices</a></li>
                                                        <li><a href="education.html" class="nav-link">Education and Communication</a></li>
                                                        <li><a href="sustainability.html" class="nav-link">Sustainability</a></li>
                                                        <li><a href="mycoexpo.html" class="nav-link">MycoExpo</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="human_practices.html">Additional Excellence</a>
                                                    <ul class="submenu">
                                                        <li><a href="inclusivity.html" class="nav-link">Inclusivity</a></li>
                                                        <li><a href="entrepreneurship.html" class="nav-link">Entrepreneurship</a></li>
                                                        <li><a href="branding.html" class="nav-link">Branding and Graphic Design</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="team.html">Team</a>
                                                    <ul class="submenu">
                                                      <li><a href="team.html" class="nav-link">Members</a></li>
                                                      <li><a href="attributions.html" class="nav-link">Attributions</a></li>
                                                      <li><a href="collaborations.html" class="nav-link">Collaborations</a></li>
                                                      <li><a href="sponsors.html" class="nav-link">Sponsors</a></li>
                                                      <li><a href="https://igem.org/2021_Judging_Form?id=3979" class="nav-link">Judging Form</a></li>
                                                    </ul>
                                                </li>
                                            </ul>
                                        </nav>
                                    </div>
                                    <!-- Header-btn -->
                                    <!--<div class="header-right-btn d-none d-lg-block ml-20">
                                        <a href="contact.html" class="btn header-btn">Tap</a>
                                    </div>-->
                                </div>
                            </div>
                            <!-- Mobile Menu -->
                            <div class="col-12">
                                <div class="mobile_menu d-block d-lg-none"></div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Header End -->
    </header>
    <main>
        <!--? Hero Start -->
        <div class="slider-area2">
            <div class="slider-height2 d-flex align-items-center">
                <div class="container">
                    <div class="row">
                        <div class="col-xl-12">
                            <div class="hero-cap hero-cap2 pt-70">
                                <h2>Engineering Success</h2>
                                <nav aria-label="breadcrumb">
                                    <ol class="breadcrumb">
                                      <li class="breadcrumb-item"><a href="index.html">Description</a></li>
                                      <li class="breadcrumb-item"><a href="#">Design</a></li>
                                      <li class="breadcrumb-item"><a href="#">Implementation</a></li>
                                      <li class="breadcrumb-item"><a href="#">Contribution</a></li>
                                      <li class="breadcrumb-item"><a href="#">Future Directions</a></li>
                                    </ol>
                                </nav>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Hero End -->
        <!--? Our Cases Start -->
        <div class="our-cases-area section-padding30">
            <div class="container">
                <div class="row justify-content-center">
                    <div class="col-xl-10 col-lg-10 ">
                        <!-- Section Tittle -->
                        <div class="section-tittle text-left mb-30">
                            <h2>In-vitro Modeling</h2>
                            <p class="pl-20 pr-20">The putative chimeric chitinases designed by the team is a novel protein designed by orchestrating the best attributes of the wild-type chitinases. To analyze the efficiency of the designed protein structures to chitin polymer, we simulated the molecular docking process to calculate its binding affinity.</p>
                        </div>
                    </div>
                </div>
                <div class="section-top-border">
                  <h2 class="mb-30">Methods and Servers</h3>
                  <div class="row">
                    <div class="col-md-6">
                      <div class="single-defination">
                        <h3 class="mb-20">Swiss Model</h4>
                        <p><a class="text-warning" href="https://swissmodel.expasy.org/interactive" color="blue">SWISS-MODEL</a> is a web-based server specializing in homology-based protein structure modeling. It is used to generate reliable structural information of the novel chimeric protein.  A lot of development has taken place in the software since its development 28 years ago. It helps users with minimal computational expertise to generate reliable protein models and easily visualize and interpret the results. As more structures are being experimentally determined, it has been noticed that the surfaces that interact with the protein's counterpart are often conserved among structures that share homology.</p>
                      </div>
                    </div>
                    <div class="col-md-6">
                      <div class="single-defination">
                        <h3 class="mb-20">Raptor X</h4>
                        <p><a class="text-warning" href="http://raptorx.uchicago.edu" color="blue">Raptor X</a> is a 3D structure prediction method based on template-based modeling, homology modeling, and protein threading. This method can be implemented to make it easy to turn on and off specific features and specify which pairwise relationships should be predicted. This method works well for distantly related proteins and proteins with very few annotated sequence profiles.</p>
                      </div>
                    </div>
                  </div>
                </div>
                <div class="visit-tailor-area fix">
                    <div class="section-tittle text-center justify mb-30">
                        <span>Our Approach</span>
                        <p>We proceeded with modeling a 3-Dimensional using a fully automated protein structure homology-based modeling server namely SWISS-MODEL. Using the above-mentioned server we were successful in getting a predictive structure for our chimeric proteins but due to the fact that the predictive structures were based on homology, some parts of the protein were truncated and it showed us fragments of the final protein based on similar proteins already annotated. This led us to use another contact prediction server developed by the Xu group, excelling at tertiary and contact prediction for protein sequences without close homologs in the Protein Data Bank (PDB) called Raptor X. The RaptorX server could not provide the predicted structure of the chitinase protein from <i>Pseudoalteromonas</i>sp DL-6. This could be due to the large nucleotide sequence of the protein. The protein structures predicted by RaptorX have been visualized using UCSF Chimera.</p>
                    </div>
                </div>
                <br>
                <br>
                <h2 class="mb-30">RaptorX Protein Structures</h3>
											<div class="row gallery-item text-center">
                        <div class="col-md-3">
                          <iframe class="img-fluid" src="assets/img/drylab/BC1.mp4"></iframe>
                          <p>BC1</p>
                        </div>
                        <div class="col-md-3">
                          <iframe class="img-fluid" src="assets/img/drylab/BC2.mp4"></iframe>
                          <p>BC2</p>
                        </div>
                        <div class="col-md-3">
                          <iframe class="img-fluid" src="assets/img/drylab/PC1.mp4"></iframe>
                          <p>PC1</p>
                        </div>
                        <div class="col-md-3">
                          <iframe class="img-fluid" src="assets/img/drylab/PC2.mp4"></iframe>
                          <p>PC2</p>
                        </div>
											</div>
                  <p class="pl-20 pr-20">To find the effective binding of the chitin polymer to the chimeric chitinase protein compared to the wild-type chitinases, AutoDock software (Autodock tools-1.5.6 and AutoDock Vina) was used. It is a molecular modeling simulation software and is especially effective for protein-ligand docking. Here the ligand used was Chitin octamer (CID 24978517) since no other long-chain chitin polymer was present in the online repositories (visualised in UCSF Chimera).</p>
                  <div class="image-block, text-center">
                    <img src="assets/img/drylab/structural_model_01.png" alt="sample image" class="img-fluid">
                    <p><strong>Figure 1.</strong> Chitin octamer (CID 24978517)</p>
                  </div>
                  <p class="pl-20 pr-20">The protein structures were prepared before docking by removing water molecules, adding hydrogen atoms, and adding Kollman charges. These modifications are necessary for the efficient binding of the ligand to the protein through non-covalent interactions.</p>
                  <p class="pl-20 pr-20">The software was run using the structures determined by the RaptorX server and the data was as follows:</p>
                  <br>
                  <br>
                  <h2 class="mb-30">Autodock Result of BC1</h3>
                  <table class="table table-hover table-bordered">
                    <thead>
                <tr>
                  <th scope="col">Mode</th>
                  <th scope="col">Affinity (kcal/mol)</th>
                  <th scope="col">Dist from RMSD L.B.</th>
                  <th scope="col">Best mode RMSD U.B.</th>
               </tr>
              </thead>
              <tbody>
                <tr>
                <td><p>1</p></td>
                <td><p>-16.5</p></td>
                <td><p>0</p></td>
                <td><p>0</p></td>
                </tr>

                <tr>
                <td><p>2</p></td>
                <td><p>-15.8</p></td>
                <td><p>7.224</p></td>
                <td><p>14.44</p></td>
                </tr>

                <tr>
                <td><p>3</p></td>
                <td><p>-15.4</p></td>
                <td><p>1.212</p></td>
                <td><p>2.709</p></td>
                </tr>

                <tr>
                <td><p>4</p></td>
                <td><p>-15</p></td>
                <td><p>21.146</p></td>
                <td><p>26.965</p></td>
                </tr>

                <tr>
                <td><p>5</p></td>
                <td><p>-15</p></td>
                <td><p>21.907</p></td>
                <td><p>26.347</p></td>
                </tr>

                <tr>
                <td><p>6</p></td>
                <td><p>-14.8</p></td>
                <td><p>24.365</p></td>
                <td><p>27.715</p></td>
                </tr>

                <tr>
                <td><p>7</p></td>
                <td><p>-14.7</p></td>
                <td><p>5.631</p></td>
                <td><p>16.924</p></td>
                </tr>

                <tr>
                <td><p>8</p></td>
                <td><p>-14.6</p></td>
                <td><p>6.803</p></td>
                <td><p>12.991</p></td>
                </tr>

                <tr>
                <td><p>9</p></td>
                <td><p>-14.5</p></td>
                <td><p>8.903</p></td>
                <td><p>15.797</p></td>
                </tr>
                </tbody>
                </table>

                </tbody>
                </table>

                <br>
                <br>
                <h2 class="mb-30">Autodock Result of BC2</h3>
                <table class="table table-hover table-bordered">
            <thead>
              <tr>
                <th scope="col">Mode</th>
                <th scope="col">Affinity (kcal/mol)</th>
                <th scope="col">Dist from RMSD L.B.</th>
                <th scope="col">Best mode RMSD U.B.</th>
             </tr>
            </thead>
            <tbody>
              <tr>
              <td><p>1</p></td>
              <td><p>-11.8</p></td>
              <td><p>0</p></td>
              <td><p>0</p></td>
              </tr>

              <tr>
              <td><p>2</p></td>
              <td><p>-11.7</p></td>
              <td><p>1.287</p></td>
              <td><p>2.467</p></td>
              </tr>

              <tr>
              <td><p>3</p></td>
              <td><p>-10.8</p></td>
              <td><p>6.245</p></td>
              <td><p>14.777</p></td>
              </tr>

              <tr>
              <td><p>4</p></td>
              <td><p>-10.8</p></td>
              <td><p>2.900</p></td>
              <td><p>10.137</p></td>
              </tr>

              <tr>
              <td><p>5</p></td>
              <td><p>-10.7</p></td>
              <td><p>3.885</p></td>
              <td><p>10.571</p></td>
              </tr>

              <tr>
              <td><p>6</p></td>
              <td><p>-10.6</p></td>
              <td><p>3.525</p></td>
              <td><p>8.198</p></td>
              </tr>

              <tr>
              <td><p>7</p></td>
              <td><p>-10.6</p></td>
              <td><p>7.961</p></td>
              <td><p>14.579</p></td>
              </tr>

              <tr>
              <td><p>8</p></td>
              <td><p>-10.6</p></td>
              <td><p>4.779</p></td>
              <td><p>10.805</p></td>
              </tr>

              <tr>
              <td><p>9</p></td>
              <td><p>-10.6</p></td>
              <td><p>33.198</p></td>
              <td><p>39.540</p></td>
              </tr>
              </tbody>
              </table>

              <br>
              <br>
              <h2 class="mb-30">Autodock Result of PC1</h3>
              <table class="table table-hover table-bordered">
            <thead>
              <tr>
                <th scope="col">Mode</th>
                <th scope="col">Affinity (kcal/mol)</th>
                <th scope="col">Dist from RMSD L.B.</th>
                <th scope="col">Best mode RMSD U.B.</th>
             </tr>
            </thead>
            <tbody>
              <tr>
              <td><p>1</p></td>
              <td><p>-14.8</p></td>
              <td><p>0</p></td>
              <td><p>0</p></td>
              </tr>

              <tr>
              <td><p>2</p></td>
              <td><p>-14.6</p></td>
              <td><p>20.789</p></td>
              <td><p>30.321</p></td>
              </tr>

              <tr>
              <td><p>3</p></td>
              <td><p>-14.5</p></td>
              <td><p>3.308</p></td>
              <td><p>7.029</p></td>
              </tr>

              <tr>
              <td><p>4</p></td>
              <td><p>-14.3</p></td>
              <td><p>2.898</p></td>
              <td><p>8.011</p></td>
              </tr>

              <tr>
              <td><p>5</p></td>
              <td><p>-14.3</p></td>
              <td><p>21.191</p></td>
              <td><p>30.614</p></td>
              </tr>

              <tr>
              <td><p>6</p></td>
              <td><p>-14.2</p></td>
              <td><p>3.355</p></td>
              <td><p>7.831</p></td>
              </tr>

              <tr>
              <td><p>7</p></td>
              <td><p>-14</p></td>
              <td><p>3.151</p></td>
              <td><p>8.385</p></td>
              </tr>

              <tr>
              <td><p>8</p></td>
              <td><p>-14</p></td>
              <td><p>2.015</p></td>
              <td><p>2.781</p></td>
              </tr>

              <tr>
              <td><p>9</p></td>
              <td><p>-14</p></td>
              <td><p>20.074</p></td>
              <td><p>27.084</p></td>
              </tr>
              </tbody>
              </table>

              <br>
              <br>
              <h2 class="mb-30">Autodock Result of PC2</h3>
                <table class="table table-hover table-bordered">
              <thead>
                <tr>
                  <th scope="col">Mode</th>
                  <th scope="col">Affinity (kcal/mol)</th>
                  <th scope="col">Dist from RMSD L.B.</th>
                  <th scope="col">Best mode RMSD U.B.</th>
               </tr>
              </thead>
              <tbody>
                <tr>
                <td><p>1</p></td>
                <td><p>-12.4</p></td>
                 <td><p>0</p></td>
                 <td><p>0</p></td>
                </tr>

                <tr>
                <td><p>2</p></td>
                <td><p>-12.2</p></td>
                 <td><p>12.275</p></td>
                 <td><p>21.421</p></td>
                </tr>

                <tr>
                <td><p>3</p></td>
                <td><p>-12.2</p></td>
                 <td><p>33.606</p></td>
                 <td><p>37.765</p></td>
                </tr>

                <tr>
                <td><p>4</p></td>
                <td><p>-12.0</p></td>
                 <td><p>9.206</p></td>
                 <td><p>17.650</p></td>
                </tr>

                <tr>
                <td><p>5</p></td>
                <td><p>-11.9</p></td>
                 <td><p>9.028</p></td>
                 <td><p>16.936</p></td>
                </tr>

                <tr>
                <td><p>6</p></td>
                <td><p>-11.9</p></td>
                 <td><p>2.270</p></td>
                 <td><p>3.084</p></td>
                </tr>

                <tr>
                <td><p>7</p></td>
                <td><p>-11.8</p></td>
                 <td><p>7.889</p></td>
                 <td><p>15.163</p></td>
                </tr>

                <tr>
                <td><p>8</p></td>
                <td><p>-11.8</p></td>
                 <td><p>1.185</p></td>
                 <td><p>2.287</p></td>
                </tr>

                <tr>
                <td><p>9</p></td>
                <td><p>-11.8</p></td>
                 <td><p>9.041</p></td>
                 <td><p>15.674</p></td>
                </tr>
                 </tbody>
                </table>
                <br>
                <p class="pl-20 pr-20">More the negative value of the affinity (kcal/mol), higher is the binding efficiency of the protein to the chitin polymer.</p>
                <br>
                <h2>Conclusion</h2>
                <p class="pl-20 pr-20">From the affinity values, it is concluded that,</p>
                <ol class="ordered-list">
                <li>The plant chitinase combo has better binding efficiency than the wild-type chitinases.</li>
                <li>The bacterial chitinase combo 1 has better binding efficiency than the wild-type chitinase (here, <i>Serratia marcescens</i>).</li>
                <li>The bacterial chitinase combo 2 has better binding efficiency than the wild-type chitinases.</li>
                </ol>
                <p class="pl-20 pr-20">
                To visualize the better binding of the chimeric chitinases, the binding affinity data were used to calculate its dissociation constant.
                </p>
                <p class="pl-20 pr-20">
                The ligand-receptor complex concentration at any given time is given by:</p>
                \[\begin{equation}
                \frac{d[L-R]}{dt}=k_{1}([L]_{T}-[L-R])([R]_{T}-[L-R])-k_{2}[L-R]
                \end{equation}\]
                <br>
                <div class="image-block, text-center">
                  <img src="assets/img/drylab/LR_interaction.png" alt="sample image" class="img-fluid">
                  <p><strong>Figure.</strong> The ligand-receptor complex conc. was calculated using J-Sim v2 software.</p>
                </div>
                <p class="pl-20 pr-20">Hence we see a higher binding affinity, therefore higher ligand-receptor concentration at any given time, with respect to wild-type chitinase proteins.</p>
              </div>
        </div>
        <!--
        <div class="image-block, text-center">
          <img src="" alt="sample image" class="img-fluid">
          <p><strong>Figure 1.</strong> Unannotated image I hope igem folks got better stuff.</p>
        </div>
        -->
        <!--? About Area Start -->
        <!--
        <div class="our-cases-area section-padding10">
            <div class="container">
                <div class="row justify-content-center">
                    <div class="col-xl-10 col-lg-10 ">
                        <!-- Section Tittle -->
                        <!--
                        <div class="section-tittle text-left mb-30">
                                <p class="pl-20 pr-20">Quantifying the activity of the protein at different physical conditions can help in characterising it. That is why we planned to work on various enzymatic models. Moreover, as mentioned above, different fungi grow differently, one would need to know how much of the protein would be needed to have an impact on their growth. So, we sought out models for IC\(_{50}\) and worked on a proxy molecule to validate the chosen models.</p>
                                <p class="pl-20 pr-20">Given the time and effort needed to clone a gene of interest and purify the protein successfully, we need to know the optimum IPTG concentration needed to help in the induction of the protein. This would reduce the time of trial to get the right concentration of IPTG. Collaborating with Team iGEM Ecuador, we had successfully calculated the amount of IPTG that would be required for optimal protein production. Moreover, we also predicted the max amount of protein that a single <i>E. coli</i> cell would be able to produce and further went on to verify our predictions with experimental results.</p>
                                <p class="pl-20 pr-20">As mentioned in our proposed implementation, we wish to use nanoparticles (PLGA) to release our chimeric chitinase into the human body. Administering the human body with a foreign particle, requires knowing the time for which the nanoparticle remains in the body system to anticipate any possible side effects. Thus we planned to model the degradation process of the nanoparticle under physiological conditions.</p>
                                <p class="pl-20 pr-20">Thus, we have attempted to model every important step in characterizing our chimeric protein, providing promising results to further work on this project and data to help design experiments in wet lab.</p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        -->
        <!-- About Area End -->
    </main>
    <footer>
        <div class="footer-wrapper">
           <!-- Footer Start-->
           <div class="footer-area footer-padding">
               <div class="container ">
                   <div class="row justify-content-between">
                       <div class="col-xl-4 col-lg-3 col-md-8 col-sm-8">
                           <div class="single-footer-caption mb-50">
                               <div class="single-footer-caption mb-30">
                                   <!-- logo -->
                                   <div class="footer-logo mb-35">
                                       <a href="index.html"><img src="assets/img/logo/logo2_footer.png" alt=""></a>
                                   </div>
                                   <div class="footer-tittle">
                                       <div class="footer-pera">
                                           <p>The iGEM Team of IISER Thiruvananthapuram</p>
                                       </div>
                                   </div>
                                   <!-- social -->
                                   <div class="footer-social">
                                       <a href="https://twitter.com/igem_iisertvm"><i class="fab fa-twitter"></i></a>
                                       <a href="https://instagram.com/igem_iisertvm"><i class="fab fa-instagram"></i></a>
                                       <a href="https://www.youtube.com/channel/UC20hJ-yyGrhZH2PbUrRyI8w"><i class="fab fa-youtube"></i></a>
                                       <a href="https://linktr.ee/igem_iisertvm"><i class="fas fa-link"></i></a>
                                   </div>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-3 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Some Links</h4>
                                   <ul>
                                       <li><a href="#">Info 1</a></li>
                                       <li><a href="#">Info 2</a></li>
                                       <li><a href="#">Info 3</a></li>
                                       <li><a href="#">Info 4</a></li>
                                       <li><a href="#">Info 5</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-2 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>iGEM Team</h4>
                                   <ul>
                                       <li><a href="#">About</a></li>
                                       <li><a href="#">Review</a></li>
                                       <li><a href="#">Info</a></li>
                                       <li><a href="#">Human</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-3 col-lg-4 col-md-6 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Contact us</h4>
                                   <ul>
                                       <li><a href="#">igem@iisertvm.ac.in</a></li>
                                       <li><a href="#">Vithura, IISER TVM</a></li>
                                       <li><a href="#">Maruthamala PO</a></li>
                                       <li class="number"><a href="#">(+91) 8240966465</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                   </div>
               </div>
           </div>
           <!-- footer-bottom area -->
           <!--<div class="footer-bottom-area">
               <div class="container">
                   <div class="footer-border">
                       <div class="row d-flex align-items-center">
                           <div class="col-xl-12 ">
                               <div class="footer-copy-right text-center">
                                  <p>
                                      Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved.
                                  </p>
                                  </div>
                              </div>
                          </div>
                      </div>
                  </div>
              </div>-->
              <!-- Footer End-->
          </div>
      </footer>

      <!-- Scroll Up -->
      <div id="back-top" >
        <a title="Go to Top" href="#"> <i class="fas fa-level-up-alt"></i></a>
    </div>

    <!-- JS here -->

    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3.0.1/es5/tex-mml-chtml.js"></script>
    <script src="./assets/js/vendor/modernizr-3.5.0.min.js"></script>
    <!-- Jquery, Popper, Bootstrap -->
    <script src="./assets/js/vendor/jquery-1.12.4.min.js"></script>
    <script src="./assets/js/popper.min.js"></script>
    <script src="./assets/js/bootstrap.min.js"></script>
    <!-- Jquery Mobile Menu -->
    <script src="./assets/js/jquery.slicknav.min.js"></script>

    <!-- Jquery Slick , Owl-Carousel Plugins -->
    <script src="./assets/js/owl.carousel.min.js"></script>
    <script src="./assets/js/slick.min.js"></script>
    <!-- One Page, Animated-HeadLin -->
    <script src="./assets/js/wow.min.js"></script>
    <script src="./assets/js/animated.headline.js"></script>
    <script src="./assets/js/jquery.magnific-popup.js"></script>

    <!-- Date Picker -->
    <script src="./assets/js/gijgo.min.js"></script>
    <!-- Nice-select, sticky -->
    <script src="./assets/js/jquery.nice-select.min.js"></script>
    <script src="./assets/js/jquery.sticky.js"></script>
    <!-- Progress -->
    <script src="./assets/js/jquery.barfiller.js"></script>

    <!-- counter , waypoint,Hover Direction -->
    <script src="./assets/js/jquery.counterup.min.js"></script>
    <script src="./assets/js/waypoints.min.js"></script>
    <script src="./assets/js/jquery.countdown.min.js"></script>
    <script src="./assets/js/hover-direction-snake.min.js"></script>

    <!-- contact js -->
    <script src="./assets/js/contact.js"></script>
    <script src="./assets/js/jquery.form.js"></script>
    <script src="./assets/js/jquery.validate.min.js"></script>
    <script src="./assets/js/mail-script.js"></script>
    <script src="./assets/js/jquery.ajaxchimp.min.js"></script>

    <!-- Jquery Plugins, main Jquery -->
    <script src="./assets/js/plugins.js"></script>
    <script src="./assets/js/main.js"></script>

</body>
</html>
