<!doctype html>
<html class="no-js" lang="zxx">
<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>Moldemort</title>
    <meta name="description" content="">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="manifest" href="site.webmanifest">
    <link rel="shortcut icon" type="image/x-icon" href="assets/img/favicon.ico">

    <!-- CSS here -->
    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
    <link rel="stylesheet" href="assets/css/owl.carousel.min.css">
    <link rel="stylesheet" href="assets/css/slicknav.css">
    <link rel="stylesheet" href="assets/css/flaticon.css">
    <link rel="stylesheet" href="assets/css/progressbar_barfiller.css">
    <link rel="stylesheet" href="assets/css/gijgo.css">
    <link rel="stylesheet" href="assets/css/animate.min.css">
    <link rel="stylesheet" href="assets/css/animated-headline.css">
    <link rel="stylesheet" href="assets/css/magnific-popup.css">
    <link rel="stylesheet" href="assets/css/fontawesome-all.min.css">
    <link rel="stylesheet" href="assets/css/themify-icons.css">
    <link rel="stylesheet" href="assets/css/slick.css">
    <link rel="stylesheet" href="assets/css/nice-select.css">
    <link rel="stylesheet" href="assets/css/style.css">
</head>
<body>
    <!-- ? Preloader Start -->
    <div id="preloader-active">
        <div class="preloader d-flex align-items-center justify-content-center">
            <div class="preloader-inner position-relative">
                <div class="preloader-circle"></div>
                <div class="preloader-img pere-text">
                    <img src="assets/img/logo/loder.png" alt="">
                </div>
            </div>
        </div>
    </div>
    <!-- Preloader End -->
    <header>
        <!-- Header Start -->
        <div class="header-area">
            <div class="main-header ">
                <div class="header-bottom  header-sticky">
                    <div class="container-fluid">
                        <div class="row align-items-center">
                            <!-- Logo -->
                            <div class="col-xl-2 col-lg-2">
                                <div class="logo">
                                    <a href="index.html"><img src="assets/img/logo/logo.png" alt=""></a>
                                </div>
                            </div>
                            <div class="col-xl-10 col-lg-10">
                                <div class="menu-wrapper  d-flex align-items-right justify-content-end">
                                    <!-- Main-menu -->
                                    <div class="main-menu d-none d-lg-block">
                                        <nav>
                                            <ul id="navigation">
                                                <li><a href="description.html">Project</a>
                                                    <ul class="submenu">
                                                        <li><a href="description.html" class="nav-link">Description</a></li>
                                                        <li><a href="design.html" class="nav-link">Design</a></li>
                                                        <li><a href="implementation.html" class="nav-link">Implementation</a></li>
                                                        <li><a href="contribution.html" class="nav-link">Contribution</a></li>
                                                        <li><a href="future_directions.html" class="nav-link">Future Directions</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="wetlab.html">Wet Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="wetlab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="fungus_identification.html" class="nav-link">Fungus Identification</a></li>
                                                        <li><a href="experiments_results.html" class="nav-link">Experiments and Results</a></li>
                                                        <li><a href="engineering.html" class="nav-link">Engineering</a></li>
                                                        <li><a href="proof_of_concept" class="nav-link">Proof of concept</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="drylab.html">Dry Lab</a>
                                                    <ul class="submenu">
                                                        <li><a href="drylab.html" class="nav-link">Overview</a></li>
                                                        <li><a href="invitro.html" class="nav-link">In-vitro modeling</a></li>
                                                        <li><a href="mathmodel.html" class="nav-link">Mathematical modeling</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="notebook.html">Journal</a>
                                                    <ul class="submenu">
                                                        <li><a href="notebook.html" class="nav-link">Notebook</a></li>
                                                        <li><a href="protocols.html" class="nav-link">Protocols</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="parts.html">Parts</a>
                                                    <ul class="submenu">
                                                        <li><a href="parts.html" class="nav-link">Overview</a></li>
                                                        <li><a href="basic_parts.html" class="nav-link">Basic Parts</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="safety.html">Safety</a></li>
                                                <li><a href="human_practices.html">Human Practices</a>
                                                    <ul class="submenu">
                                                        <li><a href="human_practices.html" class="nav-link">Overview</a></li>
                                                        <li><a href="integrated_human_practices.html" class="nav-link">Integrated Human Practices</a></li>
                                                        <li><a href="education.html" class="nav-link">Education and Communication</a></li>
                                                        <li><a href="sustainability.html" class="nav-link">Sustainability</a></li>
                                                        <li><a href="mycoexpo.html" class="nav-link">MycoExpo</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="human_practices.html">Additional Excellence</a>
                                                    <ul class="submenu">
                                                        <li><a href="inclusivity.html" class="nav-link">Inclusivity</a></li>
                                                        <li><a href="entrepreneurship.html" class="nav-link">Entrepreneurship</a></li>
                                                        <li><a href="branding.html" class="nav-link">Branding and Graphic Design</a></li>
                                                    </ul>
                                                </li>
                                                <li><a href="team.html">Team</a>
                                                    <ul class="submenu">
                                                      <li><a href="team.html" class="nav-link">Members</a></li>
                                                      <li><a href="attributions.html" class="nav-link">Attributions</a></li>
                                                      <li><a href="collaborations.html" class="nav-link">Collaborations</a></li>
                                                      <li><a href="sponsors.html" class="nav-link">Sponsors</a></li>
                                                      <li><a href="https://igem.org/2020_Judging_Form?id=3979" class="nav-link">Judging Form</a></li>
                                                    </ul>
                                                </li>
                                            </ul>
                                        </nav>
                                    </div>
                                    <!-- Header-btn -->
                                    <!--<div class="header-right-btn d-none d-lg-block ml-20">
                                        <a href="contact.html" class="btn header-btn">Tap</a>
                                    </div>-->
                                </div>
                            </div>
                            <!-- Mobile Menu -->
                            <div class="col-12">
                                <div class="mobile_menu d-block d-lg-none"></div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Header End -->
    </header>
    <main>
        <!--? Hero Start -->
        <div class="slider-area2">
            <div class="slider-height2 d-flex align-items-center">
                <div class="container">
                    <div class="row">
                        <div class="col-xl-12">
                            <div class="hero-cap hero-cap2 pt-70">
                                <h2>Engineering Success</h2>
                                <nav aria-label="breadcrumb">
                                    <ol class="breadcrumb">
                                      <li class="breadcrumb-item"><a href="index.html">Description</a></li>
                                      <li class="breadcrumb-item"><a href="#">Design</a></li>
                                      <li class="breadcrumb-item"><a href="#">Implementation</a></li>
                                      <li class="breadcrumb-item"><a href="#">Contribution</a></li>
                                      <li class="breadcrumb-item"><a href="#">Future Directions</a></li>
                                    </ol>
                                </nav>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </div>
        <!-- Hero End -->
        <!--? Our Cases Start -->
        <div class="our-cases-area section-padding30">
            <div class="container">
                <div class="row justify-content-center">
                    <div class="col-xl-10 col-lg-10 ">
                        <!-- Section Tittle -->
                        <div class="section-tittle text-left mb-30">
                            <h2>Mathematical Modeling</h2>
                            <p class="pl-20 pr-20">Mathematical modeling is an integral part of understanding the feasibility of our project. Here, for MOLDEMORT, we have attempted to model extremely crucial components of our project using mathematical equations to predict how our chimeric protein may work in-vitro. We have provided all the equations that have been in use, along with our understanding of the requirements of the models. The data thus obtained can be further used to design experiments in the wet lab.</p>
                        </div>
                        <section class="sample-text-area">
                    			<div class="container box_1170">
                    				<h2 class="text-center">Fungal Growth Models</h2>
                            <br>
                            <p class="pl-20 pr-20">
                              The team has identified seven fungal species through morphology and Sanger sequencing. To check the activity of our chitinase enzymes, we need to determine the growth rate and lag phase of the species to compare it with the changed parameters when inflicted with the extracted protein.
                            </p>
                            <p class="pl-20 pr-20">To analyze the growth of the fungus, two different methods were used:</p>
                            <br>
                            <h3 class="text-center" class="pl-20 pr-20">Surface area based growth model</h3>
                            <br>
                    				<p class="pl-20 pr-20">
                              When grown on a Petri plate in a PDA (Potato dextrose agar) medium, fungus tends to grow radially (given no contamination). The increasing radius and hence the increasing surface area is a parameter that can be used to check its growth rate over time.<sup>[1]</sup>
                            </p>
                            <p class="pl-20 pr-20">
                              The unidentified samples were inoculated on Petri plates and observed for 4-5 days to capture images of the growing fungus. It was assumed that inoculum for all fungal species was of equal size. The images were aligned and analyzed using the ImageJ software, where the radius and the corresponding area were measured.
                            </p>
                            <div class="image-block, text-center">
                              <img src="assets/img/drylab/ImageJ_demo.png" alt="sample image" class="img-fluid">
                              <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
                            </div>
                            <p class="pl-20 pr-20">The team has considered several growth models, which were mostly used for bacterial growth, yet can be used for fungal growth as well.<sup>[2][3]</sup></p>
                            <h4 class="pl-20 pr-20">Gompertz</h4>
                            \[\begin{equation}
                            y=y_{0}+C \cdot exp\left(-exp\left(\frac{\mu}{C+1}e(\lambda-t)\right)\right)
                            \end{equation}\]
                            <h4 class="pl-20 pr-20">Logistic</h4>
                            \[\begin{equation}
                            y=y_{0}+\frac{C}{1+ exp\left(\frac{4\mu}{C+2}(\lambda-t)\right)}
                            \end{equation}
                            \]
                            <h4 class="pl-20 pr-20">Richards</h4>
                            \[\begin{equation}
                            y=y_{0}+\frac{C}{\left(1+\beta e^{1+\beta}\cdot exp\left(\frac{\mu}{C}(1+\beta)(1+\frac{1}{\beta})(\lambda-t)\right)\right)^{\frac{1}{\beta}}}
                            \end{equation}\]

                            Where,<br>
                            \(y_0\) = Initial log count or absorbance<br>
                            \(y\) = Log count or absorbance at time \(t\)<br>
                            \(\mu\) = Maximum growth rate<br>
                            \(\lambda\) = Lag time<br>
                            \(C\) = Increase in log count or absorbance from \(y_0\) to \(y_{max}\)<br>
                            \(\beta\) = Model coefficient
                            <br>
                            <br>
                            <p class="pl-20 pr-20">These three equations were used to fit the curve and find the unknown parameters. It has been observed that the Gompertz model of growth gives us the least RMSE (Root Mean Squared Error), hence proving to be the best model for our data.</p>
                    			</div>
                    		</section>
												<table class="table table-hover table-bordered">
  											<thead>
											    <tr>
											      <th scope="col">Fungus Sample</th>
											      <th scope="col">Goodness of fit</th>
											      <th scope="col">SSE</th>
											      <th scope="col">R-square</th>
														<th scope="col">Adjusted R-square</th>
														<th scope="col">RMSE</th>
														<th scope="col">\(\mu\) (max growth rate) in \(cm^2/hr\)</th>
														<th scope="col">\(\lambda\) (lag time) in \(hr\)</th>
														<th scope="col">\(\beta\)</th>
											    </tr>
											  </thead>
  											<tbody>
                          <tr>
                          <td rowspan="3" ><p><i>R. oryzae</i></p></td>
                          <td><p>Gompertz</p></td>
                          <td><p>98.28</p></td>
                          <td><p>0.9934</p></td>
                          <td><p>0.9925</p></td>
                          <td><p>3.505</p></td>
                          <td><p>3.319</p></td>
                          <td><p>23.42</p></td>
                          <td rowspan="3" ><br>
                          <p>0.2167</p></td>
                          </tr>

                          <tr>
                          <td><p>Richards</p></td>
                          <td><p>109.8</p></td>
                          <td><p>0.9926</p></td>
                          <td><p>0.9917</p></td>
                          <td><p>3.693</p></td>
                          <td><p>1.402</p></td>
                          <td><p>13.32</p></td>
                          </tr>

                          <tr>
                          <td><p>Logistic</p></td>
                          <td><p>180.2</p></td>
                          <td><p>0.9879</p></td>
                          <td><p>0.9863</p></td>
                          <td><p>4.746</p></td>
                          <td><p>3.303</p></td>
                          <td><p>14.04</p></td>
                          </tr>

                          <tr>
                          <td rowspan="3" ><p><i>A. niger</i></p></td>
                          <td><p>Gompertz</p></td>
                          <td><p>3.982</p></td>
                          <td><p>0.9982</p></td>
                          <td><p>0.9981</p></td>
                          <td><p>0.4704</p></td>
                          <td><p>0.4765</p></td>
                          <td><p>47.95</p></td>
                          <td rowspan="3" ><br>
                          <p>0.048</p></td>
                          </tr>

                          <tr>
                          <td><p>Richards</p></td>
                          <td><p>3.951</p></td>
                          <td><p>0.9982</p></td>
                          <td><p>0.998</p></td>
                          <td><p>0.4821</p></td>
                          <td><p>0.05567</p></td>
                          <td><p>26.28</p></td>
                          </tr>

                          <tr>
                          <td><p>Logistic</p></td>
                          <td><p>12.64</p></td>
                          <td><p>0.9942</p></td>
                          <td><p>0.9939</p></td>
                          <td><p>0.8378</p></td>
                          <td><p>0.4471</p></td>
                          <td><p>27.46</p></td>
                          </tr>

                          <tr>
                          <td rowspan="3" ><p><i>N. indicum</i></p></td>
                          <td><p>Gompertz</p></td>
                          <td><p>25.43</p></td>
                          <td><p>0.9966</p></td>
                          <td><p>0.9964</p></td>
                          <td><p>1.188</p></td>
                          <td><p>0.8935</p></td>
                          <td><p>65.78</p></td>
                          <td rowspan="3" ><br>
                          <p>0.2167</p></td>
                          </tr>

                          <tr>
                          <td><p>Richards</p></td>
                          <td><p>28.72</p></td>
                          <td><p>0.9962</p></td>
                          <td><p>0.996</p></td>
                          <td><p>1.263</p></td>
                          <td><p>0.3809</p></td>
                          <td><p>45.27</p></td>
                          </tr>

                          <tr>
                          <td><p>Logistic</p></td>
                          <td><p>60.19</p></td>
                          <td><p>0.992</p></td>
                          <td><p>0.9915</p></td>
                          <td><p>1.829</p></td>
                          <td><p>0.8587</p></td>
                          <td><p>46.42</p></td>
                          </tr>
											  </tbody>
												</table>
                        <p><strong>SSE:</strong> Sum of squared estimate of errors ; <strong>RMSE:</strong> Root Mean Squared Error</p>
                        <p class="pl-20 pr-20">The surface area based growth model was conducted before the completion of fungal identification. The names have been assigned after its identification.</p>
                    </div>
                </div>
                <div class="row">
                    <div class="col-lg-4 col-md-6 col-sm-6">
                        <div class="single-cases mb-40">
                            <div class="cases-img">
                                <img src="assets/img/drylab/r.oryzae_gompertz.png" alt="">
                            </div>
                            <div class="cases-caption">
                                <p>Unannotated image, not resized yet</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6 col-sm-6">
                        <div class="single-cases mb-40">
                            <div class="cases-img">
                                <img src="assets/img/drylab/a.niger_gompertz.png" alt="">
                            </div>
                            <div class="cases-caption">
                                <p>Unannotated image, not resized yet</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6 col-sm-6">
                        <div class="single-cases">
                            <div class="cases-img">
                                <img src="assets/img/drylab/n.indicum_gompertz.png" alt="">
                            </div>
                            <div class="cases-caption">
                                <p>Unannotated image, not resized yet</p>
                            </div>
                        </div>
                    </div>
                        <div class="container">
                            <div class="row justify-content-center">
                                <div class="col-xl-10 col-lg-10 ">
                    <section class="sample-text-area">
                    <div class="container box_1170">
                    <p class="pl-20 pr-20">Few drawbacks in this method:</p>
                    <ul class="ordered-list">
                          <li>The growth of the fungus is confined by the surface area of the Petri plate itself, and once the fungus covers the plate, it starts growing volumetrically, making it challenging to analyze the growth.</li>
                          <li>Most of the fungus present on our campus are sporulating. The spread of spores within the plate causes several colonies to grow together, rendering the experiment useless, as different colonies superimpose on each other, and no proper radius is visible.</li>
                          <li>Sometimes, the fungal species may not grow radially but in a non-uniform shape, increasing the manual labour of calculating the area.</li>
                    </ul>
                    </div>
                    <br>
                    <br>
                    <h3 class="text-center" class="pl-20 pr-20">OD based growth model</h3>
                    <br>
                    <p class="pl-20 pr-20">
                      Scattering of light by a replicating population over time, and hence the changing OD is a good proxy for the growth of fungus instead of measuring its changing radius.<sup>[4]</sup>
                    </p>
                    <p class="pl-20 pr-20">
                      The fungal species were grown in a 96-well plate in a PDB (Potato dextrose broth) medium, with spores suspended in 1x PBS (Phosphate buffered saline), and observations were taken 4-5 days for an interval of 6 hours at 605nm. The assumption is made that the spore suspension has been uniformly distributed throughout the well.
                    </p>
                    <div class="image-block, text-center">
                      <img src="assets/img/drylab/r.oryzae_1XPDB.png" alt="sample image" class="img-fluid">
                      <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
                    </div>
                    <div class="image-block, text-center">
                      <img src="assets/img/drylab/a.versicolor_1XPDB.png" alt="sample image" class="img-fluid">
                      <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
                    </div>
                    <table class="table table-hover table-bordered">
                    <thead>
                      <tr>
                        <th scope="col"></th>
                        <th scope="col">Growth rate (Absorbance/hr)</th>
                        <th scope="col">Lag time (hr)</th>
                      </tr>
                    </thead>
                    <tbody>
                      <tr>
                      <td><p>Rhizopus oryzae</p></td>
                      <td><p>0.3145</p></td>
                      <td><p>27.69 </p></td>
                      </tr>
                      <tr>
                      <td><p>Aspergillus versicolor</p></td>
                      <td><p>0.1701</p></td>
                      <td><p>42.72</p></td>
                      </tr>
                    </tbody>
                  </table>
                  <p class="pl-20 pr-20">Limitations of this method:</p>
                  <ul class="ordered-list">
                        <li>The 96 well plate was kept in an incubator-cum-shaker at 28 degrees celsius, but the heat inside the incubator caused the media in the plate to evaporate and dry up. Hence, a humidity chamber needs to be made to minimize the evaporation of media from the plate.</li>
                  </ul>
                  <br>
                  <br>
                  <p class="pl-20 pr-20">These growth models would be needed to compare the changing growth rate and lag time when the fungal species is administered with our designed chitinases. A slower growth rate and higher lag time would help us to support our alternate hypothesis of developing chimeric chitinases with enhanced antifungal activity than wild-type chitinases.</p>
                  <br>
                  <br>
                  <h3 class="text-center" class="pl-20 pr-20">\(IC_{50}\) of an Antifungal-Nipagin</h3>
                  <br>
                  <p class="pl-20 pr-20">
                    Nipagin, being a methylparaben, is an antifungal agent primarily used in cosmetics and food preservatives.<sup>[5]</sup>
                  </p>
                  <div class="image-block, text-center">
                    <img src="assets/img/drylab/nipagin.png" alt="sample image" class="img-fluid">
                    <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
                  </div>
                  <p class="pl-20 pr-20">
                    We carried out an antifungal assay using Nipagin to calculate its IC50 for the fungal species identified by the team. This same experiment would be repeated with the purified protein of our interest to calculate its IC50 for different fungal species. We wish to check the applicability of the models which calculate IC50 using Nipagin, which then would be used for our protein.
                  </p>
                  <p class="pl-20 pr-20">
                    IC50 of the antifungal here would indicate how much of the chemical would be needed to inhibit the in vitro growth of the given fungal species by 50% <sup>[6]</sup>. Lower the IC50, better the efficiency of the antifungal.
                  </p>
                  <p class="pl-20 pr-20">
                    An initial trial in experimenting, with antifungal concentration starting from 1000μg/mL, with serial dilutions of ½ till the final concentration of 0.488μg/mL gave us the IC50 as follows: Rhizopus oryzae (230.46μg/mL) and Aspergillus versicolor (288.916μg/mL)</p>
                  <p class="pl-20 pr-20">
                    The experiment was repeated to verify the results with different mathematical methods. Since the IC50 of Nipagin for both the fungus came in about 200-300μg/mL range, the graduation of nipagin concentration was changed to 800μg/mL to 0.5μg/mL, diluting it by 100μg/mL for the initial concentrations.
                  </p>
                  <p class="pl-20 pr-20">The inhibition percentage has been calculated using three different methods:<sup>[7]</sup></p>
                  <h4 class="pl-20 pr-20">Method 1</h4>
                  \[\begin{equation}
                  I=\frac{I_{max}-I_{min}}{1+\left(\frac{conc}{IC_{50}}\right)^\gamma}+I_{max}
                  \end{equation}\]
                  Where, \(\gamma\) = Steepness constant
                  <br>
                  <br>
                  <h4 class="pl-20 pr-20">Method 2</h4>
                  \[\begin{equation}
                  I=I_{min}+\left(\frac{I_{max}\cdot conc}{(IC_{50})^\gamma+(conc)^\gamma}\right)^\gamma
                  \end{equation}
                  \]
                  Where, \(\gamma\) = Sigmoidal coefficient
                  <br>
                  <br>
                  <h4 class="pl-20 pr-20">Method 3</h4>
                  \[\begin{equation}
                  I=I_{min}+\frac{I_{max}-I_{min}}{1+10^{((conc\; -\; logIC_{50})\times \text{Hill slope})}}
                  \end{equation}
                  \]
                  <br>
                  <br>
                  <div class="image-block, text-center">
                    <img src="assets/img/drylab/IC50_A.versicolor.png" alt="sample image" class="img-fluid">
                    <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
                  </div>
                  <div class="row">
                      <div class="col-lg-4 col-md-6 col-sm-6">
                          <div class="single-cases mb-40">
                              <div class="cases-img">
                                  <img src="assets/img/drylab/IC50_A.versicolor_Method_1.png" alt="">
                              </div>
                              <div class="cases-caption">
                                  <p>Unannotated image, not resized yet</p>
                              </div>
                          </div>
                      </div>
                      <div class="col-lg-4 col-md-6 col-sm-6">
                          <div class="single-cases mb-40">
                              <div class="cases-img">
                                  <img src="assets/img/drylab/IC50_A.versicolor_Method_2.png" alt="">
                              </div>
                              <div class="cases-caption">
                                  <p>Unannotated image, not resized yet</p>
                              </div>
                          </div>
                      </div>
                      <div class="col-lg-4 col-md-6 col-sm-6">
                          <div class="single-cases">
                              <div class="cases-img">
                                  <img src="assets/img/drylab/IC50_A.versicolor_Method_3.png" alt="">
                              </div>
                              <div class="cases-caption">
                                  <p>Unannotated image, not resized yet</p>
                              </div>
                          </div>
                      </div>
                </div>
            </div>
            <table class="table table-hover table-bordered">
            <thead>
              <tr>
                <th scope="col">Method Used</th>
                <th scope="col">Hill coefficient</th>
                <th scope="col">\(IC_{50}\) (\(μg/mL\))</th>
              </tr>
            </thead>
            <tbody>
            <tr>
  <td><p>Method 1</p></td>
  <td><p>1.2785</p></td>
  <td><p>223.534</p></td>
  </tr>

  <tr>
  <td><p>Method 2</p></td>
  <td><p>1.2308</p></td>
  <td><p>241.9844</p></td>
  </tr>

  <tr>
  <td><p>Method 3</p></td>
  <td><p>1.2308</p></td>
  <td><p>241.9797</p></td>
  </tr>

  <tr>
  <td rowspan="2" ><br>
  <p>Using GraphPad</p></td>
  <td><p>-1.23</p></td>
  <td><p>241.243</p></td>
  </tr>

  <tr>
  <td colspan="2" ><p>(R2=0.8474)</p></td>
  </tr>
  </tbody>
</table>
            <div class="image-block, text-center">
              <img src="assets/img/drylab/IC50_R.oryzae.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
            </div>
          </div>
          <div class="row">
              <div class="col-lg-4 col-md-6 col-sm-6">
                  <div class="single-cases mb-40">
                      <div class="cases-img">
                          <img src="assets/img/drylab/IC50_R.oryzae_Method_1.png" alt="">
                      </div>
                      <div class="cases-caption">
                          <p>Unannotated image, not resized yet</p>
                      </div>
                  </div>
              </div>
              <div class="col-lg-4 col-md-6 col-sm-6">
                  <div class="single-cases mb-40">
                      <div class="cases-img">
                          <img src="assets/img/drylab/IC50_R.oryzae_Method_2.png" alt="">
                      </div>
                      <div class="cases-caption">
                          <p>Unannotated image, not resized yet</p>
                      </div>
                  </div>
              </div>
              <div class="col-lg-4 col-md-6 col-sm-6">
                  <div class="single-cases">
                      <div class="cases-img">
                          <img src="assets/img/drylab/IC50_R.oryzae_Method_3.png" alt="">
                      </div>
                      <div class="cases-caption">
                          <p>Unannotated image, not resized yet</p>
                      </div>
                  </div>
              </div>
        </div>
          <table class="table table-hover table-bordered">
          <thead>
            <tr>
              <th scope="col">Method Used</th>
              <th scope="col">Hill coefficient</th>
              <th scope="col">\(IC_{50}\) (\(μg/mL\))</th>
            </tr>
          </thead>
          <tbody>
            <tr>
            <td><p>Method 1</p></td>
            <td><p>2.4848</p></td>
            <td><p>231.6738</p></td>
            </tr>

            <tr>
            <td><p>Method 2</p></td>
            <td><p>2.7604</p></td>
            <td><p>294.2999</p></td>
            </tr>

            <tr>
            <td><p>Method 3</p></td>
            <td><p>2.7604</p></td>
            <td><p>294.2999</p></td>
            </tr>

            <tr>
            <td><p>Using GraphPad</p></td>
            <td><p>-2.172</p></td>
            <td><p>255.899</p></td>
            </tr>

            <tr>
            <td></td>
            <td colspan="2" ><p>(\(R^2\) = 0.9202)</p></td>
            </tr>
          </tbody>
        </table>

            <br>
            <br>
            <h3 class="text-center" class="pl-20 pr-20">Ligand-Receptor Interaction</h3>
            <br>
            <p class="pl-20 pr-20">
              Chitinase-chitin interaction can be considered as a ligand-receptor interaction, where the chitin polymer acts as a ligand that binds to the active sites of the chitinase protein, acting as receptors. The docking of the chitin octamer to the chimeric chitinases had already been demonstrated through Autodoc results, which provides us with the binding energy. The binding energy can be used to calculate the dissociation constants,Kd using the equation:
            </p>
            \[\begin{equation}
            \Delta G=RT \cdot lnK_{d}
            \end{equation}\]
            <p class="pl-20 pr-20">
              Here, the ligand is the chitin polymer, of which the cell wall of fungi is made of. The receptor is the active site of the chitinase protein. Formation of ligand-receptor complex is given by:</p>
            \[L+R \underset{k_2}{\stackrel{k_1}{\rightleftharpoons}} L-R\]
            <p class="pl-20 pr-20">
              This complex is formed by the virtue of non-covalent bonds like hydrogen bonding, electrostatic bonding, etc. The forward rate is given by \(k_1[L][R]\) and reverse rate is given by, \(k_2[L-R]\). Rate of change in the concentration of the ligand-receptor complex is given by,</p>
            \[\begin{equation}
            \frac{d[L-R]}{dt}=k_1[L][R]-k_2[L-R] \tag{1}
            \end{equation}\]
            <p class="pl-20 pr-20">
              Considering 1:1 ratio of ligand and receptor entity, we have,</p>
            \[\begin{equation}
            \frac{d[L]}{dt}=\frac{d[R]}{dt}=-k_1[L][R]+k_2[L-R]
            \end{equation}\]
            <p class="pl-20 pr-20">At the steady state,</p>
            \[\begin{equation}
            \begin{aligned}
            &\frac{d[L-R]}{dt}=0\\
            \implies & k_1[L][R]-k_2[L-R]=0\\
            \implies &\frac{k_1}{k_2}=\frac{[L-R]}{[L][R]}=k_{eq}
            \end{aligned}
            \end{equation}\]
            <p class="pl-20 pr-20">The affinity of the ligand is usually represented by the dissociation constant, \(K_d=\frac{1}{k_{eq}}\)</p>
            <p class="pl-20 pr-20">Considering the total concentration of ligand and total concentration of receptor to be constant for a small period of time, we have</p>
            \[\begin{equation}
            [L]=[L]_T-[L-R] \tag{2}
            \end{equation}\]
            \[\begin{equation}
            [R]=[R]_T-[L-R] \tag{3}
            \end{equation}\]
            <p clas=="pl-20 pr-20">Substituting [2] and [3] in equation [1], we have,</p>
            \[\begin{equation}
            \frac{d[L-R]}{dt}=k_{1}\left([L]_{T}-[L-R]\right)\left([R]_{T}-[L-R]\right)-k_{2}[L-R]
            \end{equation}\]
            Where,<br>
            \([L]\) = Ligand concentration at any time t<br>
            \([R]\) = Receptor concentration at any time t<br>
            \([L-R]\) = Ligand – Receptor complex<br>
            \([L]_T\) = Total Ligand conc. [chitin octamer in our case] (known) = 1 \(\mu m\)<br>
            \([R]_T\) = Total Receptor conc.[Chitinase in our case](known) = 1 \(\mu m\)<br>
            \(k_1\) = Rate constant for association of complex (determined from \(K_d\))<br>
            \(k_2\) = Rate constant for dissociation of complex (determined from \(K_d\))<br>
            \(K_d =\frac{k_2}{k_1}\) = Dissociation constant (determined from Autodoc protein modelling data, binding energy)
            <br>
            <br>
            <br>
            <table class="table table-hover table-bordered">
            <thead>
              <tr>
                <th scope="col">Chitinase type</th>
                <th scope="col">Ligand-Receptor complex concentration</th>
              </tr>
            </thead>
            <tbody>
            <tr>
            <td><p>Plant chitinase combo 1</p></td>
            <td><p>0.957205</p></td>
            </tr>

            <tr>
            <td><p>Plant chitinase combo 2</p></td>
            <td><p>0.94672139</p></td>
            </tr>

            <tr>
            <td><p>Barley</p></td>
            <td><p>0.93794263</p></td>
            </tr>

            <tr>
            <td><p>Wheat</p></td>
            <td><p>0.83258658</p></td>
            </tr>

            <tr>
            <td><p>Bacterial chitinase combo 1</p></td>
            <td><p>0.92756562</p></td>
            </tr>

            <tr>
            <td><p>Bacterial chitinase combo 2</p></td>
            <td><p>0.93296366</p></td>
            </tr>

            <tr>
            <td><p>Serratia marcescens</p></td>
            <td><p>0.96870234</p></td>
            </tr>

            <tr>
            <td><p>Streptomyces orientalis</p></td>
            <td><p>0.8934335</p></td>
            </tr>
            </tbody>
          </table>
          <p clas=="pl-20 pr-20">The graph has been plotted using JSim Version 2 software, by putting all the required values. The code can be found here.</p>
          <div class="image-block, text-center">
            <img src="assets/img/drylab/LR_interaction.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
          </div>
          <p clas=="pl-20 pr-20">From the graph and the concentration of the Ligand-Receptor complex, it can be concluded that the engineered chitinases have more tendency to bind to the chitin polymer than the wild-type chitinase at any given time.</p>
          <p clas=="pl-20 pr-20">Note: This model depicts the binding activity of the protein to chitin octamer; itThe backward does not show the antifungal activity of the engineered protein, as only chitin polymer is taken into consideration, rather than the complex cell wall of the fungus.</p>
          <br>
          <br>
          <div class="container">
              <div class="row justify-content-center">
                  <div class="col-xl-10 col-lg-10 ">
          <section class="sample-text-area">
          <div class="container box_1170">
            <h2 class="text-center">Protein Estimation</h2>
            <br>
            <p class="pl-20 pr-20">
              We tried to estimate the amount of IPTG that would be required for the optimal protein production so as to predict and thereby help with the experimental design.
            </p>
            <p class="pl-20 pr-20">We also predicted the max amount of protein that a single E.coli cell would be able to produce and further went on to verify our predictions with experimental results. To get IPTG vs Protein output and Time vs Max Protein output curve, we tried modeling every metabolic process that takes place inside the cell after IPTG induction that is responsible for the production of our enzyme.</p>
            <br>
            <div class="image-block, text-center">
              <img src="assets/img/drylab/protein_estimate_01.png" alt="sample image" class="img-fluid">
              <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
            </div>
            <p class="pl-20 pr-20">
              The gene of our interest is expressed in the BL21 DE(3) strain of E.coli bacteria, whose expression will be under the inducible promoter. The inducer here is IPTG (Isopropyl β-d-1-thiogalactopyranoside). The T7 RNA polymerase which binds to the promoter of our gene of interest and helps in transcription is further under the control of the Lac operon. The IPTG thus have to bind with both:</p>
              <ul class="ordered-list">
                    <li>Lac repressor produced by LacI gene in genomic DNA (to enable the transcription of T7 RNA polymerase) and</li>
                    <li>Lac repressor produced by LacI gene in plasmid DNA so as to free the site at which T7 RNA Polymerase binds and transcribes the gene of interest.</li>
              </ul>
              <br>
              <br>
            <p class="pl-20 pr-20">
              The metabolic processes for Enzyme production are described in terms of the following Ordinary differential equations (ODEs):<sup>[9]</sup>
            </p>
            <ul class="unordered-list">
              <b>For Genomic Part of E-coli</b>
           <br>
           <br>
              <li>Transcription of LacI mRNA:
               \[\begin{equation} \phi\stackrel{K_{tcI}}{\longrightarrow}mRNA_{LacI}<br>
               \end{equation}\]
                Decay of LacI mRNA:
               \[\begin{equation} mRNA_{LacI}\stackrel{K_{dmI}}{\longrightarrow}\phi<br>
               \end{equation}\]

              </li>
              \[\begin{equation}
              \frac{d[mRNA_{LacI}]}{dt}=K_{tcI}-K_{dmI}[mRNA_{LacI}]\tag{1}
              \end{equation}\]
              <li>Translation of LacI:
              \[\begin{equation}
              mRNA_{LacI}\stackrel{K_{tl}}{\longrightarrow}mRNA_{LacI}+LacI<br>
              \end{equation}\]
              Decay of LacI:
              \[\begin{equation}
              LacI\stackrel{K_{dpI}}{\longrightarrow}\phi
             \end{equation}\]
              \[\begin{equation}
              \frac{d[LacI]}{dt}=K_{tl}[mRNA_{LacI}]-K_{dpI}[LacI]\tag{2}
              \end{equation}\]
              <li>Formation of LacI and IPTG complex:
              \[\begin{equation}
              LacI+IPTG_{in} \underset{K_{bI}}{\stackrel{K_{fI}}{\rightleftharpoons}} LacI-IPTG_{in}
              \end{equation}\]
                \[\begin{equation}
                \frac{d[LacI-IPTG_{in}]}{dt}=K_{fI}[LacI][IPTG_{in}]-K_{bI}[LacI-IPTG_{in}]\tag{3}
                \end{equation}\]
              <li>Passive diffusion of IPTG across the cell membrane of E. coli:
                \[\begin{equation}
                IPTG_{ex} \underset{K_{pdiff}}{\stackrel{K_{pdiff}}{\rightleftharpoons}} IPTG_{in}
                \end{equation}\]
                \[\begin{equation}
                \frac{d[IPTG_{ex}]}{dt}=K_{pdiff}([IPTG_{ex}]-[IPTG_{in}])+K_{fdiff}([LacY-IPTG_{ex}])\tag{4}
                \end{equation}\]
              <li>Transcription of LacY mRNA:
              \[\begin{equation}
              O\stackrel{K}\longrightarrow O+mRNA_{LacY}
              \end{equation}\]
              Decay of LacY mRNA:
              \[\begin{equation}
              mRNA_{LacY}\stackrel{K_{dmY}}{\longrightarrow}\phi
              \end{equation}\]
              \[\begin{equation}
                      \frac{d[mRNA_{LacY}]}{dt}=K_{tcY}[O_T]-K_{dmY}[mRNA_{LacY}]\tag{5}
              \end{equation}\]
              <li>Translation of LacY:
               \[\begin{equation}
              mRNA_{LacY}\stackrel{K_{tlY}}{\longrightarrow}RNA_{LacY}+LacY
              \end{equation}\]
                Decay of LacY:
               \[\begin{equation}
                LacY\stackrel{K_{dY}}{\longrightarrow}\phi\
                \end{equation}\]
                \begin{equation}
                \frac{d[LacY]}{dt}=K_{tlY}[mRNA_{LacY}]+(K_{pdiff}+K_{bY})[LacY-IPTG_{ex}]-K_{fY}[LacY][IPTG_{ex}]-K_{dY}[LacY]\tag{6}
                 \end{equation}

                Formation of IPTG-LacY complex and facilitated diffusion
                \[\begin{equation}
                LacY+IPTG_{ex}\underset{K_{bY}}{\stackrel{K_{fY}}{\rightleftharpoons}}LacY-IPTG\stackrel{K_{fdiff}}{\longrightarrow}LacY+IPTG_{in}

                \end{equation}\]
              <li>Formation of \(LacY-IPTG_{ex}\) complex:
              \[\begin{equation}
              LacY+IPTG_{ex}\underset{K_{bY}}{\stackrel{K_{fY}}{\rightleftharpoons}}LacY-IPTG_{ex}
              \end{equation}\]
              </li>
                \[\begin{equation}
                \frac{d[LacY-IPTG_{ex}]}{dt}=-(K_{pdiff}+K_{bY})[LacY-IPTG]_{ex}+K_{fY}[LacY][IPTG]-K_{dY\_IPTG}[LacY-IPTG_{ex}]\tag{7}
                \end{equation}\]
              <li>Free LacI present (that is, not bounded to IPTG), which will prevent the transcription of T7 RNA polymerase<br></li>
                \[\begin{equation}
                \frac{d[LacI]_F}{dt}=(K_{tl}[mRNA_{LacI}]-K_{dpI}[LacI])-(K_{fI}[LacI][IPTG_{in}]-K_{bI}[LacI-IPTG_{in}])\tag{8}
                \end{equation}\]
                <br>
                From here, \([IPTG_{in}]\) will be replaced by \([IPTG_{Pin}]\) ; intracellular IPTG due to facilitated diffusion by LacY produced by the plasmid of E.coli due to its high copy number. The [IPTG] diffused by this virtue can bind to LacI-repressor produced by the genomic as well.
                <br>
                <br>
                <li>Production of T7 RNA polymerase mRNA:
                \[\begin{equation}
                \phi\stackrel{K_{tcT7}}{\longrightarrow}mRNA_{T7}\
                \end{equation}\]
                Decay of T7 RNA polymerase mRNA:
                 \[\begin{equation}
                 mRNA_{T7}\stackrel{K_{dmT7}}{\longrightarrow}\phi\
                 \end{equation}\]
                Binding of free LacI and \(mRNA_{T7}\):
                 \[\begin{equation}
                  LacI_{F}+mRNA_{T7}\stackrel{K_{aff}}{\longrightarrow}\phi
                  \end{equation}\]
                  </li>
                \[\begin{equation}
                      \frac{d[mRNA_{T7}]}{dt}=K_{tcT7}-K_{dmT7}[mRNA_{T7}]-K_{aff}[LacI]_F \tag{9}
                  \end{equation}\]
                  <li>T7 RNA Polymerase production :
                  \[\begin{equation}
                  mRNA_{T7}\longrightarrow{K_{tlT7}}mRNA_{T7}+T7\_RNA\_pol
                    \end{equation}\]
                   \[\begin{equation} T7\_RNA\_pol\stackrel{K_{dpT7}}{\longrightarrow}\phi
                   \end{equation}\]
                   </li>
                  \[\begin{equation}
                        \frac{d[T7\_RNA\_pol]}{dt}=K_{tlT7}[mRNA_{T7}]-K_{dpT7}[T7\_RNA\_pol] \tag{10}
                    \end{equation}\]
                  <b>For the plasmid part of the E.coli:</b>
                  <br>
                  <br>
                  <li>
                  \[\begin{equation}
                        \frac{d[mRNA_{PLacI}]}{dt}=K_{PtcI}-K_{PdmI}[mRNA_{PLacI}]\cdot (Copy\quad number) \tag{11}
                    \end{equation}\]
                    <li>

                        \[\begin{equation}
                    \frac{d[LacI_P]}{dt}=K_{Ptl}[mRNA_{PLacI}]-K_{PdpI}[LacI] \tag{12}
                    \end{equation}\]</li>
                    <li>\[\begin{equation}
                    \frac{d[LacI-IPTG_{in}]_P}{dt}=K_{PfI}[LacI_P][IPTG_{Pin}]-K_{PbI}[LacI-IPTG_{in}]_P \tag{13}
                    \end{equation}\]
                   </li>
                   <li>
                    \[\begin{equation}
                        \frac{d[IPTG_{Pin}]}{dt}=K_{Ppdiff}([IPTG]_{ex}-[IPTG_{Pin}])+K_{Pfdiff}([LacY_P-IPTG_{Pex}]) \tag{14}
                    \end{equation}\]
                   </li>
                   <li>
                    \[\begin{equation}
                    \frac{d[mRNA_{LacY}]_P}{dt}=K_{PtcY}[O_T]-K_{PdmY}[mRNA_{LacY}]_P \tag{15}
                    \end{equation}\]
                    <li>
                    \[\begin{equation}
                    \frac{d[LacY]_P}{dt}=K_{PtlY}[mRNA_{LacY}]_P+(K_{Ppdiff}+K_{PbY})[LacY\_IPTG_{ex}]_P-K_{PfY}[LacY][IPTG_{ex}]-K_{PdY}[LacY] \tag{16}
                    \end{equation}\]
                    <li>
                    \[\begin{equation}
                      \frac{d[LacY\_IPTG_{ex}]_P}{dt}=-(K_{Ppdiff}+K_{PbY})[LacY-IPTG]_{Pex}+K_{PfY}[LacY]_P[IPTG_{ex}]-K_{PdY\_IPTG}[LacY\_IPTG_{ex}]_P \tag{17}
                          \end{equation}\]
                    </li>
                    <li>

                   \[\begin{equation}
                        \frac{d[LacI]_{PF}}{dt}=(K_{Ptl}[mRNA_{LacI}]_P-K_{PdpI}[LacI]_P)-(K_{PfI}[LacI]_P[IPTG_{Pin}]-K_{PbI}[LacI-IPTG_{in}]_P) \tag{18}
                    \end{equation}\]

                    <b>Binding of T7 RNA Polymerase:</b>
                    \[\begin{equation}
                  RNA\_pol\underset{K_{ub}}{\stackrel{K_{b}}{\rightleftharpoons}}RNA\_pol_{B}
                  \end{equation}\]
                  \[\begin{equation}
                      \frac{d[T7\_RP_B]}{dt}=K_b[T7\_RNA\_pol]-K_{ub}[T7\_RP_B]-K_{aff}[LacI]_F \tag{19}
                    \end{equation}\]

                    <b>Decay of mRNA Chitinase:</b>
                    \[\begin{equation}
                  mRNA_{Chitinase}{\stackrel{K_{dmChitinase}}{\longrightarrow}}\phi
                   \end{equation}\]
                  \[\begin{equation}
                      \frac{d[mRNA_{Chitinase}]}{dt}=K_{leaky}+\frac{K_1[T7\_RP_B]^n}{[K_m]^n+[T7\_RP_B]^n}-K_{dmChitinase}[mRNA_{Chitinase}] \tag{20}
                    \end{equation}\]

                      <b>Chitinase production:</b>
                    \[\begin{equation}
                 mRNA_{Chitinase}{\stackrel{K_{tlChitinase}}{\longrightarrow}}mRNA_{Chitinase}+Chitinase
                 \end{equation}\]
                    Decay of the chitinase protein:
                    \[\begin{equation}
                    Chitinase{\stackrel{K_{dChitinase}}{\longrightarrow}}\phi
                    \end{equation}\]

                  \[\begin{equation}
                      \frac{d[Chitinase]}{dt}=K_{tlChitinase}[mRNA_{Chitinase}]-K_{dChitinase}[Chitinase]\tag{21}
                    \end{equation}\]
            </ul>
              <br>
                <br>
            <table class="table table-hover table-bordered">
            <thead>
              <tr>
                <th scope="col">Symbol</th>
                <th scope="col">Description</th>
              </tr>
            </thead>
            <tbody>
              <tr>
              <td><p>[\(mRNA_{LacI}\)]</p></td>
              <td><p>mRNA of LacI</p></td>
              </tr>

              <tr>
              <td><p>[LacI]</p></td>
              <td><p>LacI</p></td>
              </tr>

              <tr>
              <td><p>[\(IPTG_{in}\)]</p></td>
              <td><p>Intracellular IPTG</p></td>
              </tr>

              <tr>
              <td><p>[\(IPTG_{ex}\)]</p></td>
              <td><p>Extracellular IPTG</p></td>
              </tr>

              <tr>
              <td><p>[\(mRNA_{LacY}\)]</p></td>
              <td><p>mRNA of LacY</p></td>
              </tr>

              <tr>
              <td><p>[LacY]</p></td>
              <td><p>LacY</p></td>
              </tr>

              <tr>
              <td><p>[\(LacY - IPTG_{ex}\)]</p></td>
              <td><p>\(LacY - IPTG_{extracellular}\) complex</p></td>
              </tr>

              <tr>
              <td><p>[\(LacI - IPTG_{in}\)]</p></td>
              <td><p>\(LacI - IPTG_{intracellular}\) complex</p></td>
              </tr>

              <tr>
              <td><p>[\(LacI_{F}\)]</p></td>
              <td><p>Free (unbounded) LacI</p></td>
              </tr>

              <tr>
              <td><p>[\(mRNA_{\text{T7 RNA POL}}\)]</p></td>
              <td><p>mRNA of T7 RNA polymerase</p></td>
              </tr>

              <tr>
              <td><p>[T7 RNA POL]</p></td>
              <td><p>T7 RNA polymerase</p></td>
              </tr>

              <tr>
              <td><p>[T7 \(RP_{B}\)]</p></td>
              <td><p>T7 RNA polymerase bound to promoter region</p></td>
              </tr>

              <tr>
              <td><p>[\(mRNA_{\text{chitinase}}\)]</p></td>
              <td><p>mRNA of Chitinase</p></td>
              </tr>

              <tr>
              <td><p>[Chitinase]</p></td>
              <td><p>Chitinase</p></td>
              </tr>

              </tbody>
              </table>
              <p class="pl-20 pr-20">
                Note: The elements for the plasmid part are represented with the same Symbols but have an additional “P” added to them to signify plasmid.
              </p>
              <table class="table table-hover table-bordered">
              <thead>
                <tr>
                  <th scope="col">Symbol</th>
                  <th scope="col">Description</th>
                  <th scope="col">Value</th>
                </tr>
              </thead>
              <tbody>
              <tr>
              <td><p>K_{tcI}</p></td>
              <td rowspan="2" ><p>Rate constant for transcription of LacI</p></td>
              <td rowspan="2" ><p>0.23nMmin^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PtcI}</p></td>
              </tr>

              <tr>
              <td><p>K_{dmI}</p></td>
              <td rowspan="2" ><p>Degradation constant of LacI mRNA</p></td>
              <td rowspan="2" ><p>0.462 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PdmI}</p></td>
              </tr>

              <tr>
              <td><p>K_{tl}</p></td>
              <td rowspan="2" ><p>Rate constant for translation of LacI</p></td>
              <td rowspan="2" ><p>15 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{Ptl}</p></td>
              </tr>

              <tr>
              <td><p>K_{dpI}</p></td>
              <td rowspan="2" ><p>Degradation constant for LacI</p></td>
              <td rowspan="2" ><p>0.2 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PdpI}</p></td>
              </tr>

              <tr>
              <td><p>K_{pdiff}</p></td>
              <td rowspan="2" ><p>Rate constant for passive diffusion of IPTG</p></td>
              <td rowspan="2" ><p>0.92 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{Ppdiff}</p></td>
              </tr>

              <tr>
              <td><p>K_{fdiff}</p></td>
              <td rowspan="2" ><p>Rate constant of facilitated diffusion</p></td>
              <td rowspan="2" ><p>6*10^4 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{Pfdiff}</p></td>
              </tr>

              <tr>
              <td><p>K_{fY}</p></td>
              <td rowspan="2" ><p>Forward reaction rate constant for formation of LacY - IPTG complex</p></td>
              <td rowspan="2" ><p>0.12nM^(-1)min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PfY}</p></td>
              </tr>

              <tr>
              <td><p>K_{bY}</p></td>
              <td rowspan="2" ><p>Backward reaction rate constant for the formation of LacY - IPTG complex</p></td>
              <td rowspan="2" ><p>0.1 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PbY}</p></td>
              </tr>

              <tr>
              <td><p>K_{dYI}</p></td>
              <td rowspan="2" ><p>Degradation constant for LacY - IPTG complex</p></td>
              <td rowspan="2" ><p>0.2 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PdYI}</p></td>
              </tr>

              <tr>
              <td><p>K_{fI}</p></td>
              <td rowspan="2" ><p>Forward reaction rate constant for formation of LacY - IPTG(intracellular) complex</p></td>
              <td rowspan="2" ><p>0.12 nM^(-1)min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PfI}</p></td>
              </tr>

              <tr>
              <td><p>K_{bI}</p></td>
              <td rowspan="2" ><p>Backward reaction rate constant for formation of LacI - IPTG(intracellular) complex</p></td>
              <td rowspan="2" ><p>12 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PbI}</p></td>
              </tr>

              <tr>
              <td><p>K_{tcY}</p></td>
              <td rowspan="2" ><p>Rate constant for transcription of LacY</p></td>
              <td rowspan="2" ><p>0.5nMmin^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PtcY}</p></td>
              </tr>

              <tr>
              <td><p>K_{dmY}</p></td>
              <td rowspan="2" ><p>Degradation constant of LacY mRNA</p></td>
              <td rowspan="2" ><p>0.462 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PdmY}</p></td>
              </tr>

              <tr>
              <td><p>K_{tlY}</p></td>
              <td rowspan="2" ><p>Rate constant of translation of LacY</p></td>
              <td rowspan="2" ><p>30 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PtlY}</p></td>
              </tr>

              <tr>
              <td><p>K_{dY}</p></td>
              <td rowspan="2" ><p>Degradation constant for LacY</p></td>
              <td rowspan="2" ><p>0.2 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{PdY}</p></td>
              </tr>

              <tr>
              <td><p>K_{tcT7}</p></td>
              <td><p>Rate constant of transcription of T7 RNA Pol</p></td>
              <td><p>0.5 nMmin^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{dmT7}</p></td>
              <td><p>Degradation constant of T7 RNA Pol mRNA</p></td>
              <td><p>0.462 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{aff}</p></td>
              <td><p>Affinity constant of LacI to the operator region</p></td>
              <td><p>0.2 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{tlT7}</p></td>
              <td><p>Rate constant of translation of T7 RNA Pol</p></td>
              <td><p>6.116 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{dpT7}</p></td>
              <td><p>Degradation constant of T7 RNA Pol</p></td>
              <td><p>0.462 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{b}</p></td>
              <td><p>Binding constant of T7 RNA to Promoter</p></td>
              <td><p>0.22 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{ub}</p></td>
              <td><p>Unbinding constant</p></td>
              <td><p>28.8 min^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{leaky}</p></td>
              <td><p>Rate constant for leaky expression</p></td>
              <td><p>0.01 nMmin^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_m</p></td>
              <td><p>Michaelis Menten constant</p></td>
              <td><p>3*10^5 nM<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_1</p></td>
              <td><p>Transcription Rate constant of Chitinase mRNA</p></td>
              <td><p>2.3658 * 10^3 nMmin^(-1)<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{dm_chitinase}</p></td>
              <td><p>Degradation Rate constant for Chitinase mRNA</p></td>
              <td><p>0.462min^-1<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{tl_chitinase}</p></td>
              <td><p>Translation Rate constant for Chitinase</p></td>
              <td><p>10 min^-1<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              <tr>
              <td><p>K_{dp_chitinase}</p></td>
              <td><p>Degradation constant for chitinase</p></td>
              <td><p>0.2 min^-1<a href="https://pubmed.ncbi.nlm.nih.gov/19186128/" >[9]</a></p></td>
              </tr>

              </tbody>
              </table>
              <div class="image-block, text-center">
                <img src="assets/img/drylab/enzyme_conc_02.png" alt="sample image" class="img-fluid">
                <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
              </div>
              <p class="pl-20 pr-20">Through our model, we were able to predict that at the concentration of 0.1 - 1 mM IPTG, we can obtain optimal protein production. This prediction helped us in the design of experiments concerning the expression of the protein. It must be noted that these predictions are well in line with the standard procedures that are being used for IPTG induction.</p>
              <p class="pl-20 pr-20">Our other area of interest was predicting the total amount of enzyme that could be produced by a single E Coli: Through experimental data, we got to know that 2 litres of culture produce about 20 mg of protein. Assuming there are 10^10 bacteria / L [10] of culture (general assumption) we were able to calculate the amount of protein produced by a single E.coli to be ~ 256000 nM. These experimental results are very close to our prediction which is 255853.577 nM/ cell.</p>
              <div class="image-block, text-center">
                <img src="assets/img/drylab/enzyme_conc_01.png" alt="sample image" class="img-fluid">
                <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
              </div>
          </div>
        </section>
        <section class="sample-text-area">
        <div class="container box_1170">
          <h2 class="text-center">Predicting enzyme activity</h2>
          <br>
          <p class="pl-20 pr-20">
            Since our chimeric Chitinase comprises domains from two different chitinases, with both of them having a binding domain, we could, in theory,  predict the activity of our chitinase by analyzing the binding of the substrate with individual domains and further coupling the activity to get a combined activity.<sup>[11]</sup>
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/drylab/enzyme_activity.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
          </div>
          <p class="pl-20 pr-20">
            We here analyse the activity of bacterial Chitinase combo 2 consisting of domains from two chitinases - Serratia marcescens chitinase and Streptomyces orientalis chitinase. Here instead of Streptomyces orientalis we have used proxy species of the same genus Streptomyces griseus as the literature data for Streptomyces orientalis was unavailable. The dissociation constants and Vmax for both chitinases were determined from the literature survey.<sup>[12][13]</sup>
          </p>
          \[\begin{equation}
K_{D1}=\frac{[E][S]}{[ES]}\implies [ES]=\frac{[E][S]}{K_{D1}}   \tag{1}
\end{equation}\]
\[\begin{equation}
K_{D2}=\frac{[ES][S]}{[ESS]}\implies [ESS]=\frac{[ES][S]}{K_{D2}}=\frac{[E][S]^{2}}{K_{D1}K_{D2}} (Using\quad (1))
\end{equation}\]

Relative to the Total Enzyme Concentration, \(E_T\)
 \[\begin{equation}
  \frac{[ES]}{[E_{T}]}=\frac{[ES]}{[E]+[ES]+[ESS]}=\frac{\frac{[E][S]}{K_{D1}}}{[E]+\frac{[E][S]}{K_{D1}}+\frac{[E][S]^{2}}{K_{D1}K_{D2}}}=\frac{\frac{[S]}{K_{D1}}}{1+\frac{[S]}{K_{D1}}+\frac{[S]^2}{K_{D1}K_{D2}}}
\end{equation}\]

 \[\begin{equation}
\frac{[ESS]}{[E_T]}=\frac{[ESS]}{[E]+[ES]+[ESS]}=\frac{\frac{[E][S]^2}{K_{D1}K_{D2}}}{[E]+\frac{[E][S]}{K_{D1}}+\frac{[E][S]^2}{K_{D1}K_{D2}}}=\frac{\frac{[S]^2}{K_{D1}K_{D2}}}{1+\frac{[S]}{K_{D1}}+\frac{[S]^2}{K_{D1}K_{D2}}}
\end{equation}\]

The total velocity of this system is given by:<br>
 \[\begin{equation}
v=\frac{V_{max1}\frac{[S]}{K_{D1}}+V_{max2}\frac{[S]^2}{K_{D1}K_{D2}}}{1+\frac{[S]}{K_{D1}}+\frac{[S]^2}{K_{D1}K_{D2}}}
\end{equation}\]
Where,<br>
\(K_{D1}\)= Dissociation constant for chitinase of <i>Serratia marcescens</i><br>
\(K_{D2}\)= Dissociation constant for chitinase of <i>Amycolatopsis orientalis</i><br>
\(V_{max1}=K_{cat1}[E_T] and V_{max2}=K_{cat2}[E_T]\)

          <p class="pl-20 pr-20">
            The Vmax for chitinase was found to be 2.3739 mmol/min, which gives us an idea of how adding domains from high activity enzymes could in theory improve the efficiency of low activity enzymes.
          </p>
          <div class="image-block, text-center">
            <img src="assets/img/drylab/lineweaver_plot.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
          </div>
        </section>
        <section class="sample-text-area">
        <div class="container box_1170">
          <h2 class="text-center">Nanoencapsulation</h2>
          <br>
          <br>
          <p class="pl-20 pr-20">
            For the delivery of our chimeric chitinase into the human system, we are looking forward to proposing/ using nanoparticles (specifically PLGA) that are suitable due to their ability for controlled drug release, improved therapeutic efficiency, and reduction in the dose of drug administered.
          </p>
          <p class="pl-20 pr-20">
            Upon doing assays using our engineered chimeric protein, we have found that our protein is stable at room temperature and shows promising activity.Thus, it further strengthens our proposal to use PLGA as a delivery molecule to the human body.
          </p>
          <p class="pl-20 pr-20">
            PLGA - Poly(lactic-co-glycolic acid) is a synthetic copolymer made up of Lactic Acid (LA) and glycolic Acid(GA) and is extensively used for controlled drug delivery(approved by FDA). In an aqueous environment, the chain undergoes a hydrolytic attack on its ester bonds and degrades into LA and GA which are endogenous compounds normally metabolized through the Krebs cycle. By changing the LA/GA ratio and molecular weight in the polymer we could easily control the degradation time of the molecule. The degradation of the PLGA microsphere typically starts at the center of the sphere and the degradation front moves outwards.
          </p>
          <p class="pl-20 pr-20">
            Here we are trying to model and predict the degradation kinetics of PLGA (LA/GA ratio 50:50 and Mw 41.9 KDa in human body which helps us understanding the controlled drug release in human body.
          </p>
          <p class="pl-20 pr-20">
            Protein release from PLGA Microsphere(MS) is an autocatalytic polymer degradation mechanism, involving 3 prominent steps:<sup>[14]</sup>
          </p>
          <ul>
            <li>Water intruding in the device through its micro/macropores, and subsequent occurring of matrix plasticization</li>
            <li>Hydrolytically initiation of PLGA degradation at chain backbone and production of  acidic by-products</li>
            <li>Produced acids further catalyze polymer degradation</li>
          </ul>
          <p class="pl-20 pr-20">Assumptions made</p>
          <ul class="ordered-list">
            <li>Spherical symmetry</li>
            <li>Invariant geometry during protein unloading</li>
          </ul>
          <p class="pl-20 pr-20">
            PLGA degradation governs protein solubilization and release. PLGA degradation is promoted in the MS center due to the polymer autocatalytic degradation mechanism, causing a decrease in pH and enhancing the degradation process.
          </p>
          \[\begin{equation}
	\rho=\frac{r}{R_{MS}}  \tag{1}
\end{equation}\]
Where, <br>
\(\rho\)=  Non-dimensional results<br>
RMS=Average radius of Microsphere<br>
r= Generic radius within Microsphere<br>

Neglecting the initial hydration phase,<br>
\[\begin{equation}
A+P{\stackrel{k_1,k_{-1}}{\longleftarrow}}{A\cdot P{\stackrel{k_2}{\longrightarrow}}A+S  \tag{2}
\end{equation}\]
<br>
Where,<br>
A= Soluble acid (carbonic acid in present blood)<br>
P= Non-degraded polymer acting as substrate<br>
\(A\cdot P\)= Partially degraded, insoluble polymer<br>
\(k_1\)= Rate constant for forward reaction<br>
\(k_{-1}\)= Rate constant for backward reaction<br>
\(k_2\)= Rate constant for final degradation<br>

Equation (2) can be considered as a Michaelis-Menten equation. Approximating the quasi steady state, we have,<br>

\[\begin{equation}
    k_1[A][P]=(k_{-1}+k_2)[A\cdot P]   \tag{3}
\end{equation}\]

 \[\begin{equation}
	[A\cdot P]=\frac{k_1[A]_0[P]}{k_1[P]+k_{-1}+k_2}   \tag{4}
\end{equation}\]
Where,<br>
\([A]_0\)= Initial acid concentration, and \([A]+[P]=[A]_0\)

<br>
The rate of soluble product generation,<br>
\[\begin{equation}
 	\frac{d[S]}{dt}= k_2[A\cdot P]=\frac{k_1k_2[A]_0[P]}{k_1[P]+k_{-1}+k_2}  \tag{5}
\end{equation}\]

<br>
\[\begin{equation}
	\frac{d[S]}{dt}=\frac{k_0[P]}{K_M+[P]}  \tag{6}
\end{equation}\]
Where,<br>
\(K_M=\frac{(k_{-1}+k_2)}{k_1}\),  estimates the polymer-acid affinity, low KM indicates high affinity<br>
\(R_0=k_2\cdot A_0\)

<br>
\[\begin{equation}
	[S]=\frac{m_S}{V_{MS}MW_S}   \tag{7}
\end{equation}\]
Where,<br>
\(m_S\)= Mass of soluble products<br>
\(MW_S\)= Molecular weight of soluble products<br>
\(V_{MS}\)= Volume of a single MS<br>

\[\begin{equation}
	\frac{d[S]}{dt}=\frac{1}{V_{MS}MW_S}\frac{dm_S}{dt}  \tag{8}
\end{equation}\]

Substituting (8) in equation (6),
\[\begin{equation}
    \frac{dm_S}{dt}=V_{MS}MW_S\frac{R_0[P]}{K_M+[P]}  \tag{9}
\end{equation}\]
<br>
\[\begin{equation}
    [P]=\frac{m_P}{V_{MS}MW_P}   \tag{10}
\end{equation}\]
Where,<br>
\(m_P\)= Time-dependent mass of PLGA<br>
\(MW_P\) = Number-average molecular weight of PLGA<br>

Substituting (10) in equation (9),
\[\begin{equation}
\frac{dm_S}{dt}=V_{MS}MW_S\frac{R_0m_P}{K_MV_{MS}MW_P+m_P}  \tag{11}
\end{equation}\]
<br>

\[\begin{equation}
    \mu _S=\frac{m_S}{m_{MS,0}}   \tag{12}
\end{equation}\]
<br>

\[\begin{equation}
    \mu _P=\frac{m_p}{m_{MS,0}}    \tag{13}
\end{equation}\]
<br>

\(\mu_P + \mu_S=1\)

\[\begin{equation}
    \omega _P=\frac{MW_P}{MW_{P,0}}   \tag{14}
\end{equation}\]
<br>
Where,<br>
\(m_{MS,0}\)=Mass of single microspore at time 0<br>
\(MW_{P,0}\)= PLGA molecular weight at time 0<br>
\(\mu_P\)= Non-dimensional mass of residual polymer<br>
\(\mu_S\)= Non-dimensional mass of soluble products <br>
\(\omega _P\)= Non-dimensional polymer weight normalized with respect to its initial value<br>

\[\begin{equation}
    \frac{d\mu_S}{dt}=\frac{B_0(1-\mu _S)}{\psi _{\omega_{P}}+1-\mu _S}  \tag{15}
\end{equation}\]
<br>

\[\begin{equation}
    B_0=\frac{V_{MS}MW_SR_0}{m_{MS,0}}  \tag{16}
\end{equation}\]
<br>


\[\begin{equation}
    \psi=\frac{K_MV_{MS}MW_{P,0}}{m_{MS,0}}   \tag{17}
\end{equation}\]
<br>

\[\begin{equation}
    \omega _P=e^{(-k_{deg}t)}  \tag{18}
\end{equation}\]
Where,<br>
\(k_{deg}\)= Degradation constant of the polymer<br>

Polymer loss (i.e., generation of soluble acids) takes place after an induction time, \(t_{ind}\),<br>
Thus, equation (15) can be modified by including a heaviside function, as follows,<br>

\[\begin{equation}
    \frac{d\mu_S}{dt}=\frac{B_0(1-\mu _S)}{\psi _{\omega_{P}}+1-\mu _S}H(t-t_{ind})  \tag{19}
\end{equation}\]
<br>

To account for the continuous (non-discrete) transition of µS in time, Heaviside function has been mollified as shown in the following Equation
\[\begin{equation}
    \frac{d\mu_S}{dt}=\frac{1}{2}\frac{B_0(1-\mu _S)}{\psi _{\omega_{P}}+1-\mu _S}[tanh[\alpha (t-t_{ind})]+1]   \tag{20}
\end{equation}\]<br>
given the initial condition of
\[\begin{equation}
µ_S(0)=0 \tag{21}
\end{equation}\]
<br>

Assuming  density of PLGA is constant during degradation, equation (14) can be written as,<br>
\[\begin{equation}
    \mu _P=\frac{V_P}{V_{MS}}=1-R^3_f(t)   \tag{22}
\end{equation}\]
<br>

\[\begin{equation}
    R_f(t)=\sqrt[3]{\mu _S}   \tag{23}
\end{equation}\]
<br>

          <div class="image-block, text-center">
            <img src="assets/img/drylab/nano_graph.png" alt="sample image" class="img-fluid">
            <p><strong>Figure 1.</strong> Unannotated image, not resized yet</p>
          </div>
          <p class="pl-20 pr-20">
            This model predicts that our PLGA microsphere(Mw = 41.9 KDa) will undergo full disintegration in about 60 days (2 months) in physiological conditions. The rate of protein release will be in proportion with the decay rate of the microsphere and hence the dynamics of protein released could be analyzed using this model.
          </p>
        </section>
        <section class="sample-text-area">
        <div class="container box_1170">
          <h2 class="text-center">References</h2>
          <br>
          <br>
          <ul class="ordered-list">
            <li>Aunsbjerg SD, Andersen KR, Knøchel S. Real-time monitoring of fungal inhibition and morphological changes. J Microbiol Methods. 2015;119:196-202. doi:10.1016/j.mimet.2015.10.024</li>
            <li>María-Leonor Pla, Sandra Oltra, María-Dolores Esteban, Santiago Andreu, and Alfredo Palop. (2015). Comparison of Primary Models to Predict Microbial Growth by the Plate Count and Absorbance Methods.. BioMed Research International.doi.org/10.1155/2015/365025</li>
            <li>Tjørve KMC, Tjørve E (2017) The use of Gompertz models in growth analyses, and new Gompertz-model approach: An addition to the Unified-Richards family. PLoS ONE 12(6): e0178691. https://doi.org/10.1371/journal.pone.0178691</li>
            <li>Langvad F. A rapid and efficient method for growth measurement of filamentous fungi. J Microbiol Methods. 1999;37(1):97-100. doi:10.1016/s0167-7012(99)00053-6</li>
            <li>National Center for Biotechnology Information (2021). PubChem Compound Summary for CID 7456, Methylparaben. Retrieved October 8, 2021 from https://pubchem.ncbi.nlm.nih.gov/compound/Methylparaben</li>
            <li>Hoetelmans RM. "IC50 versus EC50". PK-PD relationships for anti-retroviral drugs. Amsterdam: Slotervaart Hospital. Archived from the original on 2017-05-28 – via U.S. Food and Drug Administration</li>
            <li>Sebaugh JL. Guidelines for accurate EC50/IC50 estimation. Pharm Stat. 2011;10(2):128-134. doi:10.1002/pst.426</li>
            <li>Volpe DA, Hamed SS, Zhang LK. Use of different parameters and equations for calculation of IC₅₀ values in efflux assays: potential sources of variability in IC₅₀ determination. AAPS J. 2014;16(1):172-180. doi:10.1208/s12248-013-9554-7</li>
            <li>Stamatakis M, Mantzaris NV. Comparison of deterministic and stochastic models of the lac operon genetic network [published correction appears in Biophys J. 2009 Nov 4 ;97(9):2651]. Biophys J. 2009;96(3):887-906. doi:10.1016/j.bpj.2008.10.028</li>
            <li>Benedik, Michael. (2016). Re: What is the average number of bacteria counts per mL of broth?. Retrieved from: https://www.researchgate.net/post/What-is-the-average-number-of-bacteria-counts-per-mL-of-broth/576158de217e20313c6aa5fd/citation/download</li>
            <li>https://depts.washington.edu/wmatkins/kinetics/index.html</li>
            <li>Zarei M, Aminzadeh S, Zolgharnein H, et al. Characterization of a chitinase with antifungal activity from a native Serratia marcescens B4A. Braz J Microbiol. 2011;42(3):1017-1029. doi:10.1590/S1517-838220110003000022</li>
            <li>M. Rabeeth, A. Anitha and Geetha Srikanth, 2011. Purification of an Antifungal Endochitinase from a Potential Biocontrol Agent Streptomyces griseus. Pakistan Journal of Biological Sciences, 14: 788-797.</li>
            <li>Netti PA, Biondi M, Frigione M. Experimental Studies and Modeling of the Degradation Process of Poly(Lactic-co-Glycolic Acid) Microspheres for Sustained Protein Release. Polymers. 2020; 12(9):2042. https://doi.org/10.3390/polym12092042</li>
          </ul>
        </section>
          <!-- About Area End -->
    </main>
    <footer>
        <div class="footer-wrapper">
           <!-- Footer Start-->
           <div class="footer-area footer-padding">
               <div class="container ">
                   <div class="row justify-content-between">
                       <div class="col-xl-4 col-lg-3 col-md-8 col-sm-8">
                           <div class="single-footer-caption mb-50">
                               <div class="single-footer-caption mb-30">
                                   <!-- logo -->
                                   <div class="footer-logo mb-35">
                                       <a href="index.html"><img src="assets/img/logo/logo2_footer.png" alt=""></a>
                                   </div>
                                   <div class="footer-tittle">
                                       <div class="footer-pera">
                                           <p>The iGEM Team of IISER Thiruvananthapuram</p>
                                       </div>
                                   </div>
                                   <!-- social -->
                                   <div class="footer-social">
                                       <a href="https://twitter.com/igem_iisertvm"><i class="fab fa-twitter"></i></a>
                                       <a href="https://instagram.com/igem_iisertvm"><i class="fab fa-instagram"></i></a>
                                       <a href="https://www.youtube.com/channel/UC20hJ-yyGrhZH2PbUrRyI8w"><i class="fab fa-youtube"></i></a>
                                       <a href="https://linktr.ee/igem_iisertvm"><i class="fas fa-link"></i></a>
                                   </div>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-3 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Some Links</h4>
                                   <ul>
                                       <li><a href="#">Info 1</a></li>
                                       <li><a href="#">Info 2</a></li>
                                       <li><a href="#">Info 3</a></li>
                                       <li><a href="#">Info 4</a></li>
                                       <li><a href="#">Info 5</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-2 col-lg-2 col-md-4 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>iGEM Team</h4>
                                   <ul>
                                       <li><a href="#">About</a></li>
                                       <li><a href="#">Review</a></li>
                                       <li><a href="#">Info</a></li>
                                       <li><a href="#">Human</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                       <div class="col-xl-3 col-lg-4 col-md-6 col-sm-4">
                           <div class="single-footer-caption mb-50">
                               <div class="footer-tittle">
                                   <h4>Contact us</h4>
                                   <ul>
                                       <li><a href="#">igem@iisertvm.ac.in</a></li>
                                       <li><a href="#">Vithura, IISER TVM</a></li>
                                       <li><a href="#">Maruthamala PO</a></li>
                                       <li class="number"><a href="#">(+91) 8240966465</a></li>
                                   </ul>
                               </div>
                           </div>
                       </div>
                   </div>
               </div>
           </div>
           <!-- footer-bottom area -->
           <!--<div class="footer-bottom-area">
               <div class="container">
                   <div class="footer-border">
                       <div class="row d-flex align-items-center">
                           <div class="col-xl-12 ">
                               <div class="footer-copy-right text-center">
                                  <p>
                                      Copyright &copy;<script>document.write(new Date().getFullYear());</script> All rights reserved.
                                  </p>
                                  </div>
                              </div>
                          </div>
                      </div>
                  </div>
              </div>-->
              <!-- Footer End-->
          </div>
      </footer>

      <!-- Scroll Up -->
      <div id="back-top" >
        <a title="Go to Top" href="#"> <i class="fas fa-level-up-alt"></i></a>
    </div>

    <!-- JS here -->

    <script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      "HTML-CSS": { linebreaks: { automatic: true } },
         SVG: { linebreaks: { automatic: true } }
       });
    </script>
    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3.0.1/es5/tex-mml-chtml.js"></script>
    <script src="./assets/js/vendor/modernizr-3.5.0.min.js"></script>
    <!-- Jquery, Popper, Bootstrap -->
    <script src="./assets/js/vendor/jquery-1.12.4.min.js"></script>
    <script src="./assets/js/popper.min.js"></script>
    <script src="./assets/js/bootstrap.min.js"></script>
    <!-- Jquery Mobile Menu -->
    <script src="./assets/js/jquery.slicknav.min.js"></script>

    <!-- Jquery Slick , Owl-Carousel Plugins -->
    <script src="./assets/js/owl.carousel.min.js"></script>
    <script src="./assets/js/slick.min.js"></script>
    <!-- One Page, Animated-HeadLin -->
    <script src="./assets/js/wow.min.js"></script>
    <script src="./assets/js/animated.headline.js"></script>
    <script src="./assets/js/jquery.magnific-popup.js"></script>

    <!-- Date Picker -->
    <script src="./assets/js/gijgo.min.js"></script>
    <!-- Nice-select, sticky -->
    <script src="./assets/js/jquery.nice-select.min.js"></script>
    <script src="./assets/js/jquery.sticky.js"></script>
    <!-- Progress -->
    <script src="./assets/js/jquery.barfiller.js"></script>

    <!-- counter , waypoint,Hover Direction -->
    <script src="./assets/js/jquery.counterup.min.js"></script>
    <script src="./assets/js/waypoints.min.js"></script>
    <script src="./assets/js/jquery.countdown.min.js"></script>
    <script src="./assets/js/hover-direction-snake.min.js"></script>

    <!-- contact js -->
    <script src="./assets/js/contact.js"></script>
    <script src="./assets/js/jquery.form.js"></script>
    <script src="./assets/js/jquery.validate.min.js"></script>
    <script src="./assets/js/mail-script.js"></script>
    <script src="./assets/js/jquery.ajaxchimp.min.js"></script>

    <!-- Jquery Plugins, main Jquery -->
    <script src="./assets/js/plugins.js"></script>
    <script src="./assets/js/main.js"></script>

</body>
</html>
